<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content="Pj4WWF5m9cqCcPqkGG75BI1_593zXImuFRpI8TnAous"> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> reflections on my research history | Masahiro Ono </title> <meta name="author" content="Masahiro Ono"> <meta name="description" content="reexamining treg fundmentals"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <meta property="og:site_name" content="Masahiro Ono"> <meta property="og:type" content="article"> <meta property="og:title" content="Masahiro Ono | reflections on my research history"> <meta property="og:url" content="https://monotockylab.github.io/blog/2024/reflections/"> <meta property="og:description" content="reexamining treg fundmentals"> <meta property="og:image" content="https://monotockylab.github.io/assets/img/tocky/socialmediapreview.png"> <meta property="og:locale" content="en"> <meta name="twitter:card" content="summary"> <meta name="twitter:title" content="reflections on my research history"> <meta name="twitter:description" content="reexamining treg fundmentals"> <meta name="twitter:image" content="https://monotockylab.github.io/assets/img/tocky/socialmediapreview.png"> <meta name="twitter:site" content="@MonoTockyLab"> <meta name="twitter:creator" content="@MonoTockyLab"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?fdf6bb598ff65151e77547fe2c57a41a"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://monotockylab.github.io/blog/2024/reflections/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Masahiro</span> Ono </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/researcher/">researcher </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/talks/">talks</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/research/">research</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/access/">access</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">reflections on my research history</h1> <p class="post-meta"> March 24, 2024 </p> <p class="post-tags"> <a href="/blog/2024"> <i class="fa-solid fa-calendar fa-sm"></i> 2024 </a>   ·   <a href="/blog/tag/treg"> <i class="fa-solid fa-hashtag fa-sm"></i> treg</a>   <a href="/blog/tag/thymectomy"> <i class="fa-solid fa-hashtag fa-sm"></i> thymectomy</a>   <a href="/blog/tag/motivation"> <i class="fa-solid fa-hashtag fa-sm"></i> motivation</a>     ·   <a href="/blog/category/blog"> <i class="fa-solid fa-tag fa-sm"></i> blog</a>   </p> </header> <article class="post-content"> <div id="markdown-content"> <p>Recently I was reflecting on my research journey over the past two decades, its challenges and achievements. Today, I want to highlight a crucial factor that guided my research direction and sparked my initial motivation to develop Tocky.</p> <hr> <h4 id="encountering-with-the-problem">Encountering with the Problem</h4> <p>Launching my PhD at Sakaguchi lab in 2002, I focused on GITR as a potentially useful Treg marker (note this was pre-Foxp3 era). I aimed to replicate early Treg experiments, including the milestone paper <a href="https://pubmed.ncbi.nlm.nih.gov/8760792/" rel="external nofollow noopener" target="_blank">Asano et al., J Exp Med 1996</a> from Shimon Sakaguchi’s group.</p> <p>Asanot et al. claimed that CD25+ T cells do not appear before day 3 post-birth, which assertion was pivotal as it ‘proved’ that ‘depletion of Tregs’ by day 3 thymectomy caused autoimmunity.</p> <p>I obtained a disturbing finding in 2003 (my second year in PhD): substantial numbers of Tregs existed in the neonatal spleen and thymus as early as day 1 after birth. This finding starkly contradicted the key claim of the Treg milestone paper by Asano et al., 1996.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/asano/asano1-480.webp 480w,/assets/img/asano/asano1-800.webp 800w,/assets/img/asano/asano1-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/asano/asano1.jpg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="confirming-the-non-reproducibility">Confirming the Non-Reproducibility</h4> <p>This finding was not isolated – <a href="https://www.pnas.org/doi/abs/10.1073/pnas.0403303101" rel="external nofollow noopener" target="_blank">Dujardin et al., PNAS, 2004</a> from the Bandeira group presented very similar data to what I had discovered.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/asano/asano2-480.webp 480w,/assets/img/asano/asano2-800.webp 800w,/assets/img/asano/asano2-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/asano/asano2.jpg" width="700" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="status-quo">Status Quo</h4> <p>At that time, I believed that the Treg theory, which relied heavily on the findings of Asano et al 1996, would soon be revised. Unfortunately, that has not happened.</p> <p>Many textbooks and reviews have echoed the findings of Asano et al, despite the evidence to the contrary regarding neonatal Treg dynamics. More recently, a milestone article on Treg by <a href="https://www.nature.com/articles/d42859-022-00048-z" rel="external nofollow noopener" target="_blank">Svoboda, Nature, 2022</a> still references the outdated notion as below.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/asano/asano3-480.webp 480w,/assets/img/asano/asano3-800.webp 800w,/assets/img/asano/asano3-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/asano/asano3.jpg" width="650" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="redefining-treg-research-beyond-the-boundaries">Redefining Treg Research: Beyond the Boundaries</h4> <p>This experience led me to scrutinize widely accepted papers and theories in the Treg field critically, convincing me that the Treg theories and concepts themselves needed substantial revision.</p> <p>This realization prompted my move from Japan to the UK in 2009 to extend my research and develop new tools to investigate Treg. This endevour resulted in the development of the multidimensional method Canonical Correspondence Analysis (CCA) in genomics <a class="citation" href="#RN98">(Ono et al., 2013)</a> and Tocky <a class="citation" href="#RN55">(Bending et al., 2018)</a>.</p> <hr> <h4 id="later-development">Later Development</h4> <p>The implications in the issues within Asano et al were profound. I later examined the effects of these and wrote a theoretical/opinion paper, analysing evidence from past papers and proposing a new perspective <a class="citation" href="#RN44">(Ono &amp; Tanaka, 2016)</a>.</p> <p>Further, given that the belief that Tregs constitute a unique lineage of cells originates from, and is supported by, the findings of <a href="https://pubmed.ncbi.nlm.nih.gov/8760792/" rel="external nofollow noopener" target="_blank">Asano et al.</a>, it is urgently required to establish a new research framework to investigate Foxp3 and Treg, without relying on the idea of a <em>stable lineage</em>.</p> <p>Our second Tocky publication published in the EMBO Journal <a class="citation" href="#RN56">(Bending et al., 2018)</a> investigated temporal dynamics of Foxp3 transcription and thereby addressed this challenge.</p> <hr> </div> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN55" class="col-sm-8"> <div class="title">A timer for analyzing temporally dynamic changes in transcription during differentiation in vivo</div> <div class="author"> David Bending , Paz Prieto Martı́n , Alina Paduraru , and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Catherine Ducker, Erik Marzaganov, Marie Laviron, Satsuki Kitano, Hitoshi Miyachi, Tessa Crompton, Masahiro Ono' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Journal of Cell Biology</em>, 2018 </div> <div class="periodical"> <em style="color: darkgrey;">The foundational publication introducing Tocky technology by the Ono lab, marking a breakthrough in T cell and B cell studies.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://rupress.org/jcb/article/217/8/2931/39442/A-timer-for-analyzing-temporally-dynamic-changes" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Understanding the mechanisms of cellular differentiation is challenging because differentiation is initiated by signaling pathways that drive temporally dynamic processes, which are difficult to analyze in vivo. We establish a new tool, Timer of cell kinetics and activity (Tocky; or toki [time in Japanese]). Tocky uses the fluorescent Timer protein, which spontaneously shifts its emission spectrum from blue to red, in combination with computer algorithms to reveal the dynamics of differentiation in vivo. Using a transcriptional target of T cell receptor (TCR) signaling, we establish Nr4a3-Tocky to follow downstream effects of TCR signaling. Nr4a3-Tocky reveals the temporal sequence of events during regulatory T cell (Treg) differentiation and shows that persistent TCR signals occur during Treg generation. Remarkably, antigen-specific T cells at the site of autoimmune inflammation also show persistent TCR signaling. In addition, by generating Foxp3-Tocky, we reveal the in vivo dynamics of demethylation of the Foxp3 gene. Thus, Tocky is a tool for cell biologists to address previously inaccessible questions by directly revealing dynamic processes in vivo.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN56" class="col-sm-8"> <div class="title">A temporally dynamic Foxp3 autoregulatory transcriptional circuit controls the effector Treg programme</div> <div class="author"> David Bending , Alina Paduraru , Catherine B Ducker , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Paz Prieto Martı́n, Tessa Crompton, Masahiro Ono' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>The EMBO journal</em>, 2018 </div> <div class="periodical"> <em style="color: darkgrey;">The second Tocky paper from the Ono lab uncovers the temporally dynamic regulation of Foxp3 transcription, offering new insights into T cell regulation.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.embopress.org/doi/full/10.15252/embj.201899013" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Regulatory T cells (Treg) are negative regulators of the immune response; however, it is poorly understood whether and how Foxp3 transcription is induced and regulated in the periphery during T‐cell responses. Using Foxp3‐Timer of cell kinetics and activity (Tocky) mice, which report real‐time Foxp3 expression, we show that the flux of new Foxp3 expressors and the rate of Foxp3 transcription are increased during inflammation. These persistent dynamics of Foxp3 transcription determine the effector Treg programme and are dependent on a Foxp3 autoregulatory transcriptional circuit. Persistent Foxp3 transcriptional activity controls the expression of coinhibitory molecules, including CTLA‐4 and effector Treg signature genes. Using RNA‐seq, we identify two groups of surface proteins based on their relationship to the temporal dynamics of Foxp3 transcription, and we show proof of principle for the manipulation of Foxp3 dynamics by immunotherapy: new Foxp3 flux is promoted by anti‐TNFRII antibody, and high‐frequency Foxp3 expressors are targeted by anti‐OX40 antibody. Collectively, our study dissects time‐dependent mechanisms behind Foxp3‐driven T‐cell regulation and establishes the Foxp3‐Tocky system as a tool to investigate the mechanisms behind T‐cell immunotherapies.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN44" class="col-sm-8"> <div class="title">Controversies concerning thymus‐derived regulatory T cells: fundamental issues and a new perspective</div> <div class="author"> Masahiro Ono , and Reiko J Tanaka </div> <div class="periodical"> <em>Immunology and cell biology</em>, 2016 </div> <div class="periodical"> <em style="color: darkgrey;">A landmark opinion piece challenging the reproducibility of a foundational Treg experiment, while introducing a groundbreaking dynamic view on Foxp3-mediated T cell regulation.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://onlinelibrary.wiley.com/doi/10.1038/icb.2015.65" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Thymus-derived regulatory T cells (Tregs) are considered to be a distinct T-cell lineage that is genetically programmed and specialised for immunosuppression. This perspective is based on the key evidence that CD25+ Tregs emigrate to neonatal spleen a few days later than other T cells and that thymectomy of 3-day-old mice depletes Tregs only, causing autoimmune diseases. Although widely believed, the evidence has never been reproduced as originally reported, and some studies indicate that Tregs exist in neonates. Thus we examine the consequences of the controversial evidence, revisit the fundamental issues of Tregs and thereby reveal the overlooked relationship of T-cell activation and Foxp3-mediated control of the T-cell system. Here we provide a new model of Tregs and Foxp3, a feedback control perspective, which views Tregs as a component of the system that controls T-cell activation, rather than as a distinct genetically programmed lineage. This perspective provides new insights into the roles of self-reactivity, T cell–antigen-presenting cell interaction and T-cell activation in Foxp3-mediated immune regulation.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN98" class="col-sm-8"> <div class="title">Visualising the cross-level relationships between pathological and physiological processes and gene expression: analyses of haematological diseases</div> <div class="author"> Masahiro Ono , Reiko Tanaka , Manabu Kano , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Toshio Sugiman' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PLoS One</em>, 2013 </div> <div class="periodical"> <em style="color: darkgrey;">The pioneering study developing Canonical Correspondence Analysis (CCA) as a genomics method, establishing a novel approach for transcriptome analysis.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/23301083" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>The understanding of pathological processes is based on the comparison between physiological and pathological conditions, and transcriptomic analysis has been extensively applied to various diseases for this purpose. However, the way in which the transcriptomic data of pathological cells relate to the transcriptomes of normal cellular counterparts has not been fully explored, and may provide new and unbiased insights into the mechanisms of these diseases. To achieve this, it is necessary to develop a method to simultaneously analyse components across different levels, namely genes, normal cells, and diseases. Here we propose a multidimensional method that visualises the cross-level relationships between these components at three different levels based on transcriptomic data of physiological and pathological processes, by adapting Canonical Correspondence Analysis, which was developed in ecology and sociology, to microarray data (CCA on Microarray data, CCAM). Using CCAM, we have analysed transcriptomes of haematological disorders and those of normal haematopoietic cell differentiation. First, by analysing leukaemia data, CCAM successfully visualised known relationships between leukaemia subtypes and cellular differentiation, and their characteristic genes, which confirmed the relevance of CCAM. Next, by analysing transcriptomes of myelodysplastic syndromes (MDS), we have shown that CCAM was effective in both generating and testing hypotheses. CCAM showed that among MDS patients, high-risk patients had transcriptomes that were more similar to those of both haematopoietic stem cells (HSC) and megakaryocyte-erythroid progenitors (MEP) than low-risk patients, and provided a prognostic model. Collectively, CCAM reveals hidden relationships between pathological and physiological processes and gene expression, providing meaningful clinical insights into haematological diseases, and these could not be revealed by other univariate and multivariate methods. Furthermore, CCAM was effective in identifying candidate genes that are correlated with cellular phenotypes of interest. We expect that CCAM will benefit a wide range of medical fields.</p> </div> </div> </div> </li></ol> </div> <br> <hr> <br> <ul class="list-disc pl-8"></ul> <h2 class="text-3xl font-semibold mb-4 mt-12">Enjoy Reading This Article?</h2> <p class="mb-2">Here are some more articles you might like to read next:</p> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/cca/">journey to tocky via cca</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/timer/">timer and tocky</a> </li> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Masahiro Ono. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?5d75c11f89cd96294bf5e6dd1ee1bb30"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-VJHT0Y5RYF"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-VJHT0Y5RYF");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>