<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content="Pj4WWF5m9cqCcPqkGG75BI1_593zXImuFRpI8TnAous"> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> journey to tocky via cca | Masahiro Ono </title> <meta name="author" content="Masahiro Ono"> <meta name="description" content="cca was indeed essential for tocky."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <meta property="og:site_name" content="Masahiro Ono"> <meta property="og:type" content="article"> <meta property="og:title" content="Masahiro Ono | journey to tocky via cca"> <meta property="og:url" content="https://monotockylab.github.io/blog/2024/cca/"> <meta property="og:description" content="cca was indeed essential for tocky."> <meta property="og:image" content="assets/img/cca/tockycca.jpeg"> <meta property="og:image:url" content="assets/img/cca/tockycca.jpeg"> <meta property="og:image:secure_url" content="assets/img/cca/tockycca.jpeg"> <meta property="og:locale" content="en"> <meta name="twitter:card" content="summary_large_image"> <meta name="twitter:site" content="@MonoTockyLab"> <meta name="twitter:creator" content="@MonoTockyLab"> <meta name="twitter:title" content="Masahiro Ono | journey to tocky via cca"> <meta name="twitter:description" content="cca was indeed essential for tocky."> <meta name="twitter:image" content="https://monotockylab.github.io/assets/img/MonoTockyLab.png"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?fdf6bb598ff65151e77547fe2c57a41a"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://monotockylab.github.io/blog/2024/cca/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Masahiro Ono </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/tocky/">Tocky </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/researcher/">researcher </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/talks/">talks</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/research/">research</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/japanese/">日本語</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/access/">access</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">journey to tocky via cca</h1> <p class="post-meta"> March 29, 2024 </p> <p class="post-tags"> <a href="/blog/2024"> <i class="fa-solid fa-calendar fa-sm"></i> 2024 </a>   ·   <a href="/blog/tag/cca"> <i class="fa-solid fa-hashtag fa-sm"></i> cca</a>   <a href="/blog/tag/multidimensional"> <i class="fa-solid fa-hashtag fa-sm"></i> multidimensional</a>   <a href="/blog/tag/motivation"> <i class="fa-solid fa-hashtag fa-sm"></i> motivation</a>     ·   <a href="/blog/category/blog"> <i class="fa-solid fa-tag fa-sm"></i> blog</a>   </p> </header> <article class="post-content"> <div id="markdown-content"> <h4 id="transitioning-from-molecular-immunology-to-systems-immunology">Transitioning from Molecular Immunology to Systems Immunology</h4> <p>Until 2007, I focused on the molecular biology of Treg, publishing seminal papers for the Foxp3-Runx1 interaction <a class="citation" href="#RN23">(Ono et al., 2007)</a>.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/nature-480.webp 480w,/assets/img/cca/nature-800.webp 800w,/assets/img/cca/nature-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/nature.jpg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <p>However, upon realising the fundamental problem with Treg, I chose an unconventional path - extending my expertise to computational biology💻, which would become pivotal in developing Tocky.</p> <hr> <h4 id="first-independent-studies-in-kyoto">First Independent Studies in Kyoto</h4> <p>As I was promoted to Assistant Professsor in 2007, I secured grant support from <a href="https://www.katokinen.or.jp/info/4_2english.html" rel="external nofollow noopener" target="_blank">the Kato Foundation</a> in 2008 for my first transcriptome analysis using microarray 🧬.</p> <p>However, the complexity of the data was overwhelming, challenging the simplistic analytical methods of the time.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/kato-480.webp 480w,/assets/img/cca/kato-800.webp 800w,/assets/img/cca/kato-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/kato.jpg" width="350" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="the-lack-of-appropriate-methods">The Lack of Appropriate Methods</h4> <p>Back then in 2006 - 2007, immunology papers mainly used simple methods like <a href="https://pubmed.ncbi.nlm.nih.gov/16772372/" rel="external nofollow noopener" target="_blank">log2 fold change only</a>, which was not convincing to me.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/sugimoto-480.webp 480w,/assets/img/cca/sugimoto-800.webp 800w,/assets/img/cca/sugimoto-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/sugimoto.jpeg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="exploration-and-collaborations-in-kyoto">Exploration and Collaborations in Kyoto</h4> <p>Thus, in 2008, I aimed to develop more sophisticated methods for immunological genomic analysis, identifying Principal Component Analysis (PCA) as a promising candidate 🤔🔍.</p> <p>In 2008, <a href="https://www.twitter.com/Dr_Kano" rel="external nofollow noopener" target="_blank">Dr Manabu Kano</a>, an expert in PCA in Eng. Dept, Kyoto Univ, provided invaluable guidance, teaching me mathematical foundations of PCA in data analysis.</p> <p>This collaboration was a pivotal step in my journey to redefine Treg in a data-oriented manner🎓💡.</p> <p>In addition, <a href="https://www.twitter.com/toshsugi" rel="external nofollow noopener" target="_blank">Prof Toshio Sugiman</a>, an expert in psychometrics🧠📈, introduced me to Hayashi’s Quantification Method III, a tool for quantifying <a href="https://www.jstage.jst.go.jp/article/acrt1992/1/1/1_1_17/_article" rel="external nofollow noopener" target="_blank">group relationships in psychology</a>.</p> <p>Hayashi’s statistical philosophy profoundly inspired me, and this will lead to the development of the groundbreaking method five years later.</p> <hr> <h4 id="extending-research-fields-from-kyoto-to-the-uk">Extending Research Fields: From Kyoto to the UK.</h4> <p>Aiming to adapt Hayashi’s method and PCA for genomic data in immunology, I obtained an HFSP fellowship and thereby moved from Kyoto🇯🇵 to UCL🇬🇧 , eventually led to <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0053544" rel="external nofollow noopener" target="_blank">the development of Canonical Correspondence Analysis, originally by ter Braak, for genomics</a> <a class="citation" href="#RN98">(Ono et al., 2013)</a>.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/plos-480.webp 480w,/assets/img/cca/plos-800.webp 800w,/assets/img/cca/plos-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/plos.jpeg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <p>🔬Canonical Correspondence Analysis is an efficient method for [qunatitative anlaysis of cell phenotype and gene activities, focusing on certain biological processes]((https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-1028). In addition, it can classify cells using a machine learning approach <a class="citation" href="#RN97">(Ono et al., 2014)</a>🧬💻.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/genomics-480.webp 480w,/assets/img/cca/genomics-800.webp 800w,/assets/img/cca/genomics-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/genomics.jpeg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> <h4 id="the-journey-to-tocky-is-through-cca">The Journey to Tocky is Through CCA</h4> <p>Importantly, CCA was utilized to identify <a href="https://rupress.org/jcb/article/217/8/2931/39442/A-timer-for-analyzing-temporally-dynamic-changes" rel="external nofollow noopener" target="_blank">Nr4a3 as the best correlate to TCR signalling</a>, which was a key step in developing Nr4a3-Tocky <a class="citation" href="#RN55">(Bending et al., 2018)</a> 🐭🔬.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/cca/tockycca-480.webp 480w,/assets/img/cca/tockycca-800.webp 800w,/assets/img/cca/tockycca-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/cca/tockycca.jpeg" width="500" height="auto" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr> </div> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN55" class="col-sm-8"> <div class="title">A timer for analyzing temporally dynamic changes in transcription during differentiation in vivo</div> <div class="author"> David Bending , Paz Prieto Martı́n , Alina Paduraru , Catherine Ducker , Erik Marzaganov , Marie Laviron , Satsuki Kitano , Hitoshi Miyachi , Tessa Crompton , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Journal of Cell Biology</em>, 2018 </div> <div class="periodical"> <em style="color: darkgrey;">The foundational publication introducing Tocky technology by the Ono lab, marking a breakthrough in T cell and B cell studies.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://rupress.org/jcb/article/217/8/2931/39442/A-timer-for-analyzing-temporally-dynamic-changes" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Understanding the mechanisms of cellular differentiation is challenging because differentiation is initiated by signaling pathways that drive temporally dynamic processes, which are difficult to analyze in vivo. We establish a new tool, Timer of cell kinetics and activity (Tocky; or toki [time in Japanese]). Tocky uses the fluorescent Timer protein, which spontaneously shifts its emission spectrum from blue to red, in combination with computer algorithms to reveal the dynamics of differentiation in vivo. Using a transcriptional target of T cell receptor (TCR) signaling, we establish Nr4a3-Tocky to follow downstream effects of TCR signaling. Nr4a3-Tocky reveals the temporal sequence of events during regulatory T cell (Treg) differentiation and shows that persistent TCR signals occur during Treg generation. Remarkably, antigen-specific T cells at the site of autoimmune inflammation also show persistent TCR signaling. In addition, by generating Foxp3-Tocky, we reveal the in vivo dynamics of demethylation of the Foxp3 gene. Thus, Tocky is a tool for cell biologists to address previously inaccessible questions by directly revealing dynamic processes in vivo.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN97" class="col-sm-8"> <div class="title">Visualisation of the T cell differentiation programme by Canonical Correspondence Analysis of transcriptomes</div> <div class="author"> <em>Masahiro Ono</em>, Reiko Tanaka , and Manabu Kano </div> <div class="periodical"> <em>BMC Genomics</em>, 2014 </div> <div class="periodical"> <em style="color: darkgrey;">This study establishes CCA as a vital quantitative tool for transcriptome analysis in immunological datasets.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/25428805" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>BACKGROUND: Currently, in the era of post-genomics, immunology is facing a challenging problem to translate mutant phenotypes into gene functions based on high-throughput data, while taking into account the classifications and functions of immune cells, which requires new methods. RESULTS: Here we propose a novel application of a multidimensional analysis, Canonical Correspondence Analysis (CCA), to reveal the molecular characteristics of undefined cells in terms of cellular differentiation programmes by analysing two transcriptomic datasets. Using two independent datasets, whether RNA-seq or microarray data, CCA successfully visualised the cross-level relationships between genes, cells, and differentiation programmes, and thereby identified the immunological features of mutant cells (Gata3-KO T cells and Stat3-KO T cells) in a data-oriented manner. With a new concept, differentiation variable, CCA provides an automatic classification of cell samples, which had a high sensitivity and a comparable performance to other classification methods. In addition, we elaborate how CCA results can be interpreted, and reveal the features of CCA in comparison with other visualisation techniques. CONCLUSIONS: CCA is a visualisation tool with a classification ability to reveal the cross-level relationships of genes, cells and differentiation programmes. This can be used for characterising the functional defect of cells of interest (e.g. mutant cells) in the context of cellular differentiation. The proposed approach fits with common hypothesis-oriented studies in immunology, and can be used for a wide range of molecular and genomic studies on cellular differentiation mechanisms.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN98" class="col-sm-8"> <div class="title">Visualising the cross-level relationships between pathological and physiological processes and gene expression: analyses of haematological diseases</div> <div class="author"> <em>Masahiro Ono</em>, Reiko Tanaka , Manabu Kano , and Toshio Sugiman </div> <div class="periodical"> <em>PLoS One</em>, 2013 </div> <div class="periodical"> <em style="color: darkgrey;">The pioneering study developing Canonical Correspondence Analysis (CCA) as a genomics method, establishing a novel approach for transcriptome analysis.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/23301083" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>The understanding of pathological processes is based on the comparison between physiological and pathological conditions, and transcriptomic analysis has been extensively applied to various diseases for this purpose. However, the way in which the transcriptomic data of pathological cells relate to the transcriptomes of normal cellular counterparts has not been fully explored, and may provide new and unbiased insights into the mechanisms of these diseases. To achieve this, it is necessary to develop a method to simultaneously analyse components across different levels, namely genes, normal cells, and diseases. Here we propose a multidimensional method that visualises the cross-level relationships between these components at three different levels based on transcriptomic data of physiological and pathological processes, by adapting Canonical Correspondence Analysis, which was developed in ecology and sociology, to microarray data (CCA on Microarray data, CCAM). Using CCAM, we have analysed transcriptomes of haematological disorders and those of normal haematopoietic cell differentiation. First, by analysing leukaemia data, CCAM successfully visualised known relationships between leukaemia subtypes and cellular differentiation, and their characteristic genes, which confirmed the relevance of CCAM. Next, by analysing transcriptomes of myelodysplastic syndromes (MDS), we have shown that CCAM was effective in both generating and testing hypotheses. CCAM showed that among MDS patients, high-risk patients had transcriptomes that were more similar to those of both haematopoietic stem cells (HSC) and megakaryocyte-erythroid progenitors (MEP) than low-risk patients, and provided a prognostic model. Collectively, CCAM reveals hidden relationships between pathological and physiological processes and gene expression, providing meaningful clinical insights into haematological diseases, and these could not be revealed by other univariate and multivariate methods. Furthermore, CCAM was effective in identifying candidate genes that are correlated with cellular phenotypes of interest. We expect that CCAM will benefit a wide range of medical fields.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN23" class="col-sm-8"> <div class="title">Foxp3 controls regulatory T-cell function by interacting with AML1/Runx1</div> <div class="author"> <em>Masahiro Ono</em>, Hiroko Yaguchi , Naganari Ohkura , Issay Kitabayashi , Yuko Nagamura , Takashi Nomura , Yoshiki Miyachi , Toshihiko Tsukada , and Shimon Sakaguchi </div> <div class="periodical"> <em>Nature</em>, 2007 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.nature.com/articles/nature05673" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Naturally arising CD25+CD4+ regulatory T cells (T(R) cells) are engaged in the maintenance of immunological self-tolerance and immune homeostasis by suppressing aberrant or excessive immune responses, such as autoimmune disease and allergy. T(R) cells specifically express the transcription factor Foxp3, a key regulator of T(R)-cell development and function. Ectopic expression of Foxp3 in conventional T cells is indeed sufficient to confer suppressive activity, repress the production of cytokines such as interleukin-2 (IL-2) and interferon-gamma (IFN-gamma), and upregulate T(R)-cell-associated molecules such as CD25, cytotoxic T-lymphocyte-associated antigen-4, and glucocorticoid-induced TNF-receptor-family-related protein. However, the method by which Foxp3 controls these molecular events has yet to be explained. Here we show that the transcription factor AML1 (acute myeloid leukaemia 1)/Runx1 (Runt-related transcription factor 1), which is crucially required for normal haematopoiesis including thymic T-cell development, activates IL-2 and IFN-gamma gene expression in conventional CD4+ T cells through binding to their respective promoters. In natural T(R) cells, Foxp3 interacts physically with AML1. Several lines of evidence support a model in which the interaction suppresses IL-2 and IFN-gamma production, upregulates T(R)-cell-associated molecules, and exerts suppressive activity. This transcriptional control of T(R)-cell function by an interaction between Foxp3 and AML1 can be exploited to control physiological and pathological T-cell-mediated immune responses.</p> </div> </div> </div> </li></ol> </div> <br> <hr> <br> <ul class="list-disc pl-8"></ul> <h2 class="text-3xl font-semibold mb-4 mt-12">Enjoy Reading This Article?</h2> <p class="mb-2">Here are some more articles you might like to read next:</p> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2025/Asano/">The Dogma of Treg and The True Origins of Tocky</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/timer/">timer and tocky</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/CancerImmunotherapy/">tocky illuminates mechanisms of immune checkpoint blockade</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/reflections/">reflections on my research history</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/dermatology/">professionalism and official account integrity</a> </li> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Masahiro Ono. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: April 17, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?5d75c11f89cd96294bf5e6dd1ee1bb30"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-VJHT0Y5RYF"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-VJHT0Y5RYF");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>