<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content="Pj4WWF5m9cqCcPqkGG75BI1_593zXImuFRpI8TnAous"> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Masahiro Ono </title> <meta name="author" content="Masahiro Ono"> <meta name="description" content="Publication – Dr Masahiro Ono is a T-cell expert and the inventor of the innovative Timer-of-Cell-Kinetics-and-Activity (Tocky) technology."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <meta property="og:site_name" content="Masahiro Ono"> <meta property="og:type" content="website"> <meta property="og:title" content="Masahiro Ono | publications"> <meta property="og:url" content="https://monotockylab.github.io/publications/"> <meta property="og:description" content="Publication – Dr Masahiro Ono is a T-cell expert and the inventor of the innovative Timer-of-Cell-Kinetics-and-Activity (Tocky) technology."> <meta property="og:image" content="assets/img/publication/bookroom.jpg"> <meta property="og:image:url" content="assets/img/publication/bookroom.jpg"> <meta property="og:image:secure_url" content="assets/img/publication/bookroom.jpg"> <meta property="og:locale" content="en"> <meta name="twitter:card" content="summary_large_image"> <meta name="twitter:site" content="@MonoTockyLab"> <meta name="twitter:creator" content="@MonoTockyLab"> <meta name="twitter:title" content="Masahiro Ono | publications"> <meta name="twitter:description" content="Publication – Dr Masahiro Ono is a T-cell expert and the inventor of the innovative Timer-of-Cell-Kinetics-and-Activity (Tocky) technology."> <meta name="twitter:image" content="https://monotockylab.github.io/assets/img/MonoTockyLab.png"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.png?fdf6bb598ff65151e77547fe2c57a41a"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://monotockylab.github.io/publications/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Masahiro Ono </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">research </a> </li> <li class="nav-item "> <a class="nav-link" href="/tocky/">Tocky </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/researcher/">researcher </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/talks/">talks</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/research/">research</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/japanese/">日本語</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/access/">access</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">Publication – Dr Masahiro Ono is a T-cell expert and the inventor of the innovative Timer-of-Cell-Kinetics-and-Activity (Tocky) technology.</p> </header> <article> <p>Key publications by Masahiro Ono, showcasing innovative approaches in immunological research and beyond.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication/bookroom-480.webp 480w,/assets/img/publication/bookroom-800.webp 800w,/assets/img/publication/bookroom-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/publication/bookroom.jpg" width="300" height="auto" data-zoomable="" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="tockyprep2025" class="col-sm-8"> <div class="title">TockyPrep: data preprocessing methods for flow cytometric fluorescent timer analysis</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> 2025 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1186/s12859-025-06058-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Fluorescent Timer proteins, which display time-dependent changes in their emission spectra, are invaluable for analyzing the temporal dynamics of cellular events at the single-cell level. We previously developed the Timer-of-cell-kinetics-and-activity (Tocky) tools, utilizing a specific Timer protein, Fast-FT, to monitor temporal changes in cellular activities. Despite their potential, the analysis of Timer fluorescence in flow cytometry is frequently compromised by variability in instrument settings and the absence of standardized preprocessing methods. The development and implementation of effective data preprocessing methods remain to be achieved.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="tockydev2024" class="col-sm-8"> <div class="title">A multidimensional toolkit for elucidating temporal trajectories in cell development in vivo</div> <div class="author"> <em>Masahiro Ono</em>, and Tessa Crompton </div> <div class="periodical"> <em>Development</em>, Nov 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1242/dev.204255" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Progenitor cells initiate development upon receiving key signals, dynamically altering gene and protein expression to diverge into various lineages and fates. Despite the use of several experimental approaches, including the Fluorescent Timer-based method Timer-of-cell-kinetics-and-activity (Tocky), analysing time-dependent processes at the single-cell level in vivo remains challenging. This study introduces a novel integrated experimental and computational approach, using an advanced multidimensional toolkit. This toolkit facilitates the simultaneous examination of temporal progression and T-cell profiles using high-dimensional flow cytometric data. Employing novel algorithms based on Canonical Correspondence Analysis (CCA) and network analysis, our toolkit identifies developmental trajectories and analyses dynamic changes in developing cells. The efficacy of this approach is demonstrated through analysing thymic T-cells from Nr4a3-Tocky mice, which monitor activities downstream of the T-cell receptor (TCR) signal. Further validation was achieved by deleting the proapoptotic gene Bcl2l11 (Bim) in Nr4a3-Tocky mice. This revealed dynamic changes in thymic T-cells during cellular development and negative selection following TCR signalling. Overall, this study establishes a new method for analysing the temporal dynamics of individual developing cells in response to in vivo signalling cues.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="ono2024tockylocusquantitativeanalysismethods" class="col-sm-8"> <div class="title">TockyLocus: Quantitative Analysis Methods for Flow Cytometric Fluorescent Timer Data</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://arxiv.org/abs/2411.09386" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Fluorescent Timer proteins, which spontaneously change their emission spectra over time, are valuable tools for analyzing temporal changes in cellular activities at the single-cell level. Traditional analysis of Fluorescent Timer data has mostly relied on conventional flow cytometric methods, which lacks the sophistication needed for detailed quantitative analysis. Recently, we developed the Timer-of-Cell-Kinetics-and-Activity (Tocky) tools, employing transgenic reporter systems using an mCherry mutant Timer protein, Fast-FT, to analyze these changes, implementing data preprocessing methods for Timer fluorescence. Despite this advancement, the computational implementation of effective quantitative analysis methods for Fluorescent Timer data has been lacking. In this study, we introduce rigorous algorithms for a data categorization method, designated as the Tocky locus approach, which uses normalized and trigonometric-transformed Fluorescent Timer data, suitable for quantitative analysis. We found that the five-locus approach, which categorizes Timer fluorescence into five categories, optimally captures the dynamics of Timer profiles, thus enabling effective quantitative statistical analysis and biological interpretation. The current study has developed comprehensive tools for data categorization, visualization, and statistical analysis optimized to Tocky Locus data. These algorithms have been implemented in the TockyLocus package, a new R package designed to facilitate the quantitative analysis of Timer fluorescence data. This toolkit is expected to significantly enhance the utility of experimental systems using Fluorescent Timer proteins, particularly the Tocky tools, by standardizing methods for analyzing dynamic cellular events and responses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="ono2024tockyprepdatapreprocessingmethods" class="col-sm-8"> <div class="title">TockyPrep: Data Preprocessing Methods for Flow Cytometric Fluorescent Timer Analysis</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://arxiv.org/abs/2411.04111" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Background: Fluorescent Timer proteins, which display time-dependent changes in their emission spectra, are invaluable for analyzing the temporal dynamics of cellular events at the single-cell level. We previously developed the Timer-of-cell-kinetics-and-activity (Tocky) tools, utilizing a specific Timer protein, Fast-FT, to monitor temporal changes in cellular activities. Despite their potential, the analysis of Timer fluorescence in flow cytometry is frequently compromised by variability in instrument settings and the absence of standardized preprocessing methods. The development and implementation of effective data preprocessing methods remain to be achieved. Results: In this study, we introduce the R package that automates the data preprocessing of Timer fluorescence data from flow cytometry experiments for quantitative analysis at single-cell level. Our aim is to standardize Timer data analysis to enhance reproducibility and accuracy across different experimental setups. The package includes a trigonometric transformation method to elucidate the dynamics of Fluorescent Timer proteins. We have identified the normalization of immature and mature Timer fluorescence data as essential for robust analysis, clarifying how this normalization affects the analysis of Timer maturation. These preprocessing methods are all encapsulated within the TockyPrep R package. Conclusions: TockyPrep is available for distribution via GitHub at this https URL, providing tools for data preprocessing and basic visualization of Timer fluorescence data. This toolkit is expected to enhance the utility of experimental systems utilizing Fluorescent Timer proteins, including the Tocky tools.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="ono2024gatingtreepathfindinganalysisgroupspecific" class="col-sm-8"> <div class="title">GatingTree: Pathfinding Analysis of Group-Specific Effects in Cytometry Data</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> Nov 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://arxiv.org/abs/2411.00129" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Advancements in cytometry technologies have led to a remarkable increase in the number of markers that can be analyzed simultaneously, presenting significant challenges in data analysis. Traditional approaches, such as dimensional reduction techniques and computational clustering, although popular, often face reproducibility challenges due to their heavy reliance on inherent data structures, preventing direct translation of their outputs into gating strategies to be used in downstream experiments. Here we propose the novel Gating Tree methodology, a pathfinding approach that investigates the multidimensional data landscape to unravel group-specific features without the use of dimensional reduction. This method employs novel measures, including enrichment scores and gating entropy, to effectively identify group-specific features within high-dimensional cytometric datasets. Our analysis, applied to both simulated and real cytometric datasets, demonstrates that the Gating Tree not only identifies group-specific features comprehensively but also produces outputs that are immediately usable as gating strategies for unequivocally identifying cell populations. In conclusion, the Gating Tree facilitates a comprehensive analysis of the multidimensional data landscape and provides experimentalists with practical, successive gating strategies that enhance cross-experimental comparisons and downstream analyses such as flow cytometric sorting.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="10.1093/discim/kyae006" class="col-sm-8"> <div class="title">Spectrum of Treg and self-reactive T cells: single cell perspectives from old friend HTLV-1</div> <div class="author"> <em>Masahiro Ono</em>, and Yorifumi Satou </div> <div class="periodical"> <em>Discovery Immunology</em>, May 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1093/discim/kyae006" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Despite extensive regulatory T cell (Treg) research, fundamental questions on in vivo dynamics remain to be answered. The current study aims to dissect several interwoven concepts in Treg biology, highlighting the ‘self-reactivity’ of Treg and their counterparts, namely naturally-arising memory-phenotype T-cells, as a key mechanism to be exploited by a human retroviral infection. We propose the novel key concept, Periodic T cell receptor (TCR)-signalled T-cells, capturing self-reactivity in a quantifiable manner using the Nr4a3-Timer-of-cell-kinetics-and-activity (Tocky) technology. Periodic and brief TCR signals in self-reactive T-cells contrast with acute TCR signals during inflammation. Thus, we propose a new two-axis model for T-cell activation by the two types of TCR signals or antigen recognition, elucidating how Foxp3 expression and acute TCR signals actively regulate Periodic TCR-signalled T-cells. Next, we highlight an underappreciated branch of immunological research on Human T-cell Leukemia Virus type 1 (HTLV-1) that precedes Treg studies, illuminating the missing link between the viral infection, CD25, and Foxp3. Based on evidence by single-cell analysis, we show how the viral infection exploits the regulatory mechanisms for T-cell activation and suggests a potential role of periodic TCR signalling in infection and malignant transformation. In conclusion, the new perspectives and models in this study provide a working framework for investigating Treg within the self-reactive T-cell spectrum, expected to advance understanding of HTLV-1 infection, cancer, and immunotherapy strategies for these conditions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN280" class="col-sm-8"> <div class="title">HIV-Tocky system to visualize proviral expression dynamics</div> <div class="author"> Omnia Reda , Kazuaki Monde , Kenji Sugata , Akhinur Rahman , Wajihah Sakhor , Samiul Alam Rajib , Sharmin Nahar Sithi , Benjy Jek Yang Tan , Koki Niimura , Chihiro Motozono , Kenji Maeda , <em>Masahiro Ono</em>, Hiroaki Takeuchi , and Yorifumi Satou </div> <div class="periodical"> <em>Communications Biology</em>, May 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1038/s42003-024-06025-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Determinants of HIV-1 latency establishment are yet to be elucidated. HIV reservoir comprises a rare fraction of infected cells that can survive host and virus-mediated killing. In vitro reporter models so far offered a feasible means to inspect this population, but with limited capabilities to dissect provirus silencing dynamics. Here, we describe a new HIV reporter model, HIV-Timer of cell kinetics and activity (HIV-Tocky) with dual fluorescence spontaneous shifting to reveal provirus silencing and reactivation dynamics. This unique feature allows, for the first time, identifying two latent populations: a directly latent, and a recently silenced subset, with the latter having integration features suggestive of stable latency. Our proposed model can help address the heterogeneous nature of HIV reservoirs and offers new possibilities for evaluating eradication strategies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="BPPB1" class="col-sm-8"> <div class="title">Unraveling T-cell dynamics using fluorescent timer: Insights from the Tocky system</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Biophysics and Physicobiology</em>, May 2024 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.2142/biophysico.bppb-v21.s010" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Understanding the temporal dynamics of T-cell transcription is crucial for insights into immune cell function and development. In this study, we show the features of the Timer-of-Cell-Kinetics-and-Activity (Tocky) system, which enables analysis of temporal dynamics of cell activities and differentiation, leveraging Fluorescent Timer protein, which spontaneously changes its emission spectrum from blue to red fluorescence in known kinetics, as reporters. The current study examines the properties of the Tocky system, highlighting the Timer-Angle approach, which is a core algorithm of Tocky analysis and converts Timer Blue and Red fluorescence into Timer Angle and Intensity by trigonometric transformation. Importantly, Tocky analyzes time-related events within individual cells by the two phases of measurements, distinguishing between (1) the temporal sequence of cellular activities and differentiation within the time domain, and (2) the transcription frequency within the frequency domain. The transition from time measurement to frequency analysis, particularly at the Persistent locus that bridges these domains, highlights that system’s unique property in what is measured and analyzed by Tocky. Intriguingly, the sustained transcriptional activities observed in cells at the Persistent locus may have unique biological features as demonstrated in activated regulatory T-cells (Treg) and pathogenic T-cells, respectively, using Foxp3-Tocky and Nr4a3-Tocky models. In conclusion, the Tocky system can provide crucial data for advancing our understanding of T-cell dynamics and function.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN81" class="col-sm-8"> <div class="title">Single-Cell Transcriptome Analysis of Treg</div> <div class="author"> B. J. Y. Tan , M. Ono , and Y. Satou </div> <div class="periodical"> <em>Methods Mol Biol</em>, May 2023 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1007/978-1-0716-2647-4_17" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Regulatory T-cells (Treg) are considerably heterogeneous. Thymically derived Treg (tTreg) are those, which differentiate in the thymus, while peripherally derived Treg (pTreg) differentiate from peripheral mature CD4(+) T-cells. These two populations are often identified using markers such as neuropilin-1 and Helios (for tTreg) and ROR-gammat (for pTreg) in intestines (Tanoue et al., Nat Rev Immunol 16:295-309, 2016).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="M1" class="col-sm-8"> <div class="title"></div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Methods Mol Biol</em>, May 2023 </div> <div class="periodical"> <em style="color: darkgrey;">Editor</em> </div> <div class="links"> <a href="https://doi.org/10.1007/978-1-0716-2647-4_17" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN80" class="col-sm-8"> <div class="title">CD4 T cell dynamics shape the immune response to combination oncolytic herpes virus and BRAF inhibitor therapy for melanoma</div> <div class="author"> Galabina Bozhanova , Jehanne Hassan , Lizzie Appleton , Victoria Jennings , Shane Foo , Martin McLaughlin , Charleen ML Chan Wah Hak , Emmanuel C Patin , Eva Crespo-Rodriguez , Gabby Baker , Edward Armstrong , Matthew Chiu , Hardev Pandha , Adel Samson , Victoria Roulstone , Joan Kyula , Richard Vile , Fiona Errington-Mais , Malin Pedersen , Kevin Harrington , <em>Masahiro Ono</em>, and Alan Melcher </div> <div class="periodical"> <em>Journal for ImmunoTherapy of Cancer</em>, May 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="abstract hidden"> <p>Background Combination herpes simplex virus (HSV) oncolytic virotherapy and BRAF inhibitors (BRAFi) represent promising immunogenic treatments for BRAF mutant melanoma, but an improved understanding of the immunobiology of combinations is needed to improve on the benefit of immune checkpoint inhibitors (ICI).Methods Using a BRAFV600E-driven murine melanoma model, we tested the immunogenicity of HSV/BRAFi in immunocompetent C57BL mice. In addition to standard FACS analysis, we used the ‘Timer of Cell Kinetics and Activity’ system, which can analyze the temporal dynamics of different T cell subsets. This immune data was used to inform the selection of ICI for triple combination therapy, the effects of which were then further characterized using transcriptomics.Results Adding BRAFi treatment to HSV improved anti-tumor effects in vivo but not in vitro. Immune characterization showed HSV or dual therapy led to fewer intratumoral Treg, although with a more activated phenotype, together with more effector CD8 +T cells. Tocky analysis further showed that HSV/BRAFi dual treatment reduced the Tocky signal (reflecting engagement with cognate antigen), in both Treg and conventional subsets of CD4+, but not in CD8 +cells. However, a higher percentage of Treg than of conventional CD4 +maintained frequent engagement with antigens on treatment, reflecting a predominance of suppressive over effector function within the CD4 +compartment. The only T cell subset which correlated with a reduction in tumor growth was within Tocky signal positive conventional CD4+, supporting their therapeutic role. Targeting CD25 high, antigen-engaged Treg with a depleting anti-CD25 ICI, achieved complete cures in 100% of mice with triple therapy. Transcriptomic analysis confirmed reduction in Foxp3 on addition of anti-CD25 to HSV/BRAFi, as well as increases in expression of genes reflecting interferon signaling and cytotoxic activity.Conclusions Combination HSV/BRAFi is an immunogenic therapy for BRAF mutant melanoma, but cannot fully control tumors. Dual therapy results in changes in T cell dynamics within tumors, with relatively maintained antigen signaling in Treg compared with conv CD4+. Antigen-engaged CD4 +effectors correlate with tumor growth control, and depletion of Treg by addition of an anti-CD25 ICI, releasing suppression of conventional CD4 +effectors by Treg, enhances survival and activates immune signaling within tumors.All data relevant to the study are included in the article or uploaded as online supplemental information. Not applicable.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN82" class="col-sm-8"> <div class="title">HTLV-1 persistence and leukemogenesis: A game of hide-and-seek with the host immune system</div> <div class="author"> B. J. Y. Tan , K. Sugata , M. Ono , and Y. Satou </div> <div class="periodical"> <em>Front Immunol</em>, May 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.3389/fimmu.2022.991928" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Human T-cell leukemia virus type 1 (HTLV-1), a retrovirus which mainly infects CD4(+) T cells and causes adult T-cell leukemia/lymphoma (ATL), is primarily transmitted via direct cell-to-cell transmission. This feature generates a wide variety of infected clones in hosts, which are maintained via clonal proliferation, resulting in the persistence and survival of the virus. The maintenance of the pool of infected cells is achieved by sculpting the immunophenotype of infected cells and modulating host immune responses to avoid immune surveillance. Here, we review the processes undertaken by HTLV-1 to modulate and subvert host immune responses which contributes to viral persistence and development of ATL.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN83" class="col-sm-8"> <div class="title">Single-cell level temporal profiling of tumour-reactive T cells under immune checkpoint blockade</div> <div class="author"> Jehanne Hassan , Elizabeth Appleton , Bahire Kalfaoglu , Malin Pedersen , José Almeida-Santos , Hisashi Kanemaru , Nobuko Irie , Shane Foo , Omnia Reda , Benjy J.Y. Tan , Il-mi Okazaki , Taku Okazaki , Yorifumi Satou , Kevin Harrington , Alan Melcher , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>bioRxiv</em>, May 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1101/2022.07.19.500582" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>The blockade of the immune checkpoints PD-1 and CTLA-4 enhances T cell response. However, it is largely unknown how antigen-reactive T cells regulate their checkpoint expression in vivo and whether and how the checkpoint blockade can change activation dynamics of tumour-reactive T cells. To address this, here we used Nr4a3-Timer-of-cell-kinetics-and-activity (Tocky), which allows analysis of temporal changes of activated T cells following TCR signalling in vivo. By analysing melanoma-bearing Nr4a3 Tocky mice, we elucidate hidden dynamics of tumour-reactive T cells in the steady-state. Checkpoint blockade depleted highly activated effector Treg, while promoting unique effector T cell populations, and thus differentially modulating activation of tumour-reactive T cell populations. Furthermore, multidimensional analysis and seamless analysis of Tocky and scRNA-seq revealed a full spectrum of T cell dynamics in response to tumour burden and treatment with checkpoint blockade. Lastly, we propose a rational design of combinatorial therapy to further enhance T cell activities.Competing Interest StatementThe authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN91" class="col-sm-8"> <div class="title">Nur77-Tempo mice reveal T cell steady state antigen recognition</div> <div class="author"> T. A. E. Elliot , E. K. Jennings , D. A. J. Lecky , S. Rouvray , G. M. Mackie , L. Scarfe , L. Sheriff , M. Ono , K. M. Maslowski , and D. Bending </div> <div class="periodical"> <em>Discov Immunol</em>, May 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1093/discim/kyac009" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>In lymphocytes, Nr4a gene expression is specifically regulated by antigen receptor signalling, making them ideal targets for use as distal T cell receptor (TCR) reporters. Nr4a3-Timer of cell kinetics and activity (Tocky) mice are a ground-breaking tool to report TCR-driven Nr4a3 expression using Fluorescent Timer protein (FT). FT undergoes a time-dependent shift in its emission spectrum following translation, allowing for the temporal reporting of transcriptional events. Our recent work suggested that Nr4a1/Nur77 may be a more sensitive gene to distal TCR signals compared to Nr4a3, so we, therefore, generated Nur77-Timer-rapidly-expressed-in-lymphocytes (Tempo) mice that express FT under the regulation of Nur77. We validated the ability of Nur77-Tempo mice to report TCR and B cell receptor signals and investigated the signals regulating Nur77-FT expression. We found that Nur77-FT was sensitive to low-strength TCR signals, and its brightness was graded in response to TCR signal strength. Nur77-FT detected positive selection signals in the thymus, and analysis of FT expression revealed that positive selection signals are often persistent in nature, with most thymic Treg expressing FT Blue. We found that active TCR signals in the spleen are low frequency, but CD69(+) lymphoid T cells are enriched for FT Blue(+) Red(+) T cells, suggesting frequent TCR signalling. In non-lymphoid tissue, we saw a dissociation of FT protein from CD69 expression, indicating that tissue residency is not associated with tonic TCR signals. Nur77-Tempo mice, therefore, combine the temporal dynamics from the Tocky innovation with increased sensitivity of Nr4a1 to lower TCR signal strengths.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN84" class="col-sm-8"> <div class="title">Temporal profiling of CD4 T-cell activation and differentiation upon SARS-CoV-2 spike protein immunisation</div> <div class="author"> José Almeida-Santos , Rita Berkachy , Chanidapa Adele Tye , Jehanne Hassan , Bahire Kalfaoglu , Murray E. Selkirk , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>bioRxiv</em>, May 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1101/2022.07.21.500987" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>CD4 T-cells require T-cell receptor (TCR) signalling for their activation and differentiation. Foxp3+ regulatory T-cells (Treg) are dependent on TCR signals for their differentiation and suppressive function. However, it is not fully known how TCR signalling controls the differentiation of polyclonal CD4 T-cells upon antigen recognition at the single-cell level in vivo. In this study, using Nr4a3-Tocky (Timer-of-cell-kinetics-and-activity), which analyses temporal changes of antigen-reactive T-cells following TCR signalling, we investigated T-cell response to Spike protein fragments (S1a, S1b, S2a, and S2b) upon immunisation. We show that S1a and S2a induced the differentiation of PD1hiCXCR5+ T follicular helper (Tfh) cells, which is related to CD4 T-cell immunogenicity. In contrast, S1b induced CD25hiGITRhiPD-1int Treg, which intermittently received TCR signalling. Using Foxp3-Tocky, which analyses Foxp3 transcriptional dynamics, the S1b-reactive Treg sustained Foxp3 transcription over time, which is a hallmark of activated Treg. Foxp3 fate-mapping showed that the S1b-reactive Treg were derived not from pre-existing thymic Treg, suggesting Foxp3 induction in non-Treg cells. Thus, the current study reveals temporally dynamic differentiation of CD4 T-cells and Treg upon immunisation in the polyclonal TCR repertoire.Competing Interest StatementThe authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="10.1084/jem.20212391" class="col-sm-8"> <div class="title">Context-dependent effects of IL-2 rewire immunity into distinct cellular circuits</div> <div class="author"> Carly E. Whyte , Kailash Singh , Oliver T. Burton , Meryem Aloulou , Lubna Kouser , Rafael Valente Veiga , Amy Dashwood , Hanneke Okkenhaug , Samira Benadda , Alena Moudra , Orian Bricard , Stephanie Lienart , Pascal Bielefeld , Carlos P. Roca , Francisco José Naranjo-Galindo , Félix Lombard-Vadnais , Steffie Junius , David Bending , <em>Masahiro Ono</em>, Tino Hochepied , Timotheus Y.F. Halim , Susan Schlenner , Sylvie Lesage , James Dooley , and Adrian Liston </div> <div class="periodical"> <em>Journal of Experimental Medicine</em>, Jun 2022 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1084/jem.20212391" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Interleukin 2 (IL-2) is a key homeostatic cytokine, with therapeutic applications in both immunogenic and tolerogenic immune modulation. Clinical use has been hampered by pleiotropic functionality and widespread receptor expression, with unexpected adverse events. Here, we developed a novel mouse strain to divert IL-2 production, allowing identification of contextual outcomes. Network analysis identified priority access for Tregs and a competitive fitness cost of IL-2 production among both Tregs and conventional CD4 T cells. CD8 T and NK cells, by contrast, exhibited a preference for autocrine IL-2 production. IL-2 sourced from dendritic cells amplified Tregs, whereas IL-2 produced by B cells induced two context-dependent circuits: dramatic expansion of CD8+ Tregs and ILC2 cells, the latter driving a downstream, IL-5–mediated, eosinophilic circuit. The source-specific effects demonstrate the contextual influence of IL-2 function and potentially explain adverse effects observed during clinical trials. Targeted IL-2 production therefore has the potential to amplify or quench particular circuits in the IL-2 network, based on clinical desirability.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN85" class="col-sm-8"> <div class="title">HTLV-1 infection promotes excessive T cell activation and transformation into adult T cell leukemia/lymphoma</div> <div class="author"> Benjy J. Y. Tan , Kenji Sugata , Omnia Reda , Misaki Matsuo , Kyosuke Uchiyama , Paola Miyazato , Vincent Hahaut , Makoto Yamagishi , Kaoru Uchimaru , Yutaka Suzuki , Takamasa Ueno , Hitoshi Suzushima , Hiroo Katsuya , Masahito Tokunaga , Yoshikazu Uchiyama , Hideaki Nakamura , Eisaburo Sueoka , Atae Utsunomiya , <em>Masahiro Ono</em>, and Yorifumi Satou </div> <div class="periodical"> <em>The Journal of Clinical Investigation</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1172/JCI150472" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Human T cell leukemia virus type 1 (HTLV-1) mainly infects CD4+ T cells and induces chronic, persistent infection in infected individuals, with some developing adult T cell leukemia/lymphoma (ATL). HTLV-1 alters cellular differentiation, activation, and survival; however, it is unknown whether and how these changes contribute to the malignant transformation of infected cells. In this study, we used single-cell RNA-sequencing and T cell receptor–sequencing to investigate the differentiation and HTLV-1–mediated transformation of T cells. We analyzed 87,742 PBMCs from 12 infected and 3 uninfected individuals. Using multiple independent bioinformatics methods, we demonstrated the seamless transition of naive T cells into activated T cells, whereby HTLV-1–infected cells in an activated state further transformed into ATL cells, which are characterized as clonally expanded, highly activated T cells. Notably, the greater the activation state of ATL cells, the more they acquire Treg signatures. Intriguingly, the expression of HLA class II genes in HTLV-1–infected cells was uniquely induced by the viral protein Tax and further upregulated in ATL cells. Functional assays revealed that HTLV-1–infected cells upregulated HLA class II molecules and acted as tolerogenic antigen-presenting cells to induce anergy of antigen-specific T cells. In conclusion, our study revealed the in vivo mechanisms of HTLV-1–mediated transformation and immune escape at the single-cell level.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN74" class="col-sm-8"> <div class="title">Retrovirus-induced leukemia–hijack of T-cell activation mechanisms revealed by single-cell analysis</div> <div class="author"> Benjy Jek Yang Tan , Kenji Sugata , Omnia Reda , Misaki Matsuo , Kyosuke Uchiyama , Paola Miyazato , Vincent Hahaut , Makoto Yamagishi , Kaoru Uchimaru , and Yutaka Suzuki </div> <div class="periodical"> <em>bioRxiv</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN78" class="col-sm-8"> <div class="title">HTLV-1 infection promotes excessive T cell activation and transformation into adult T cell leukemia/lymphoma</div> <div class="author"> Benjy JY Tan , Kenji Sugata , Omnia Reda , Misaki Matsuo , Kyosuke Uchiyama , Paola Miyazato , Vincent Hahaut , Makoto Yamagishi , Kaoru Uchimaru , and Yutaka Suzuki </div> <div class="periodical"> <em>Journal of Clinical Investigation</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN73" class="col-sm-8"> <div class="title">Restoring control over autoimmunity by inducing Foxp3</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Nature Immunology</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.nature.com/articles/nature05673" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p></p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN72" class="col-sm-8"> <div class="title">The pioneer transcription factors Foxa1 and Foxa2 regulate alternative RNA splicing during thymocyte positive selection</div> <div class="author"> Ching-In Lau , Jasmine Rowell , Diana C Yanez , Anisha Solanki , Susan Ross , <em>Masahiro Ono</em>, and Tessa Crompton </div> <div class="periodical"> <em>Development</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN67" class="col-sm-8"> <div class="title">T-cell dysregulation in COVID-19</div> <div class="author"> Bahire Kalfaoglu , José Almeida-Santos , Chanidapa Adele Tye , Yorifumi Satou , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Biochemical and biophysical research communications</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN70" class="col-sm-8"> <div class="title">Application of dual Nr4a1-GFP Nr4a3-Tocky reporter mice to study T cell receptor signaling by flow cytometry</div> <div class="author"> Emma K Jennings , David AJ Lecky , <em>Masahiro Ono</em>, and David Bending </div> <div class="periodical"> <em>STAR protocols</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN75" class="col-sm-8"> <div class="title">Brief homogeneous TCR signals instruct common iNKT progenitors whose effector diversification is characterized by subsequent cytokine signaling</div> <div class="author"> Sabrina Bortoluzzi , Nyambayar Dashtsoodol , Thomas Engleitner , Christoph Drees , Sabine Helmrath , Jonas Mir , Albulena Toska , Michael Flossdorf , Rupert Öllinger , and Maria Solovey </div> <div class="periodical"> <em>Immunity</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN79" class="col-sm-8"> <div class="title">Kickstarting immunity in cold tumours: localised tumour therapy combinations with immune checkpoint blockade</div> <div class="author"> Elizabeth Appleton , Jehanne Hassan , Charleen Chan Wah Hak , Nanna Sivamanoharan , Anna Wilkins , Adel Samson , <em>Masahiro Ono</em>, Kevin J Harrington , Alan Melcher , and Erik Wennerberg </div> <div class="periodical"> <em>Frontiers in Immunology</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN77" class="col-sm-8"> <div class="title">NF-κB activation in cardiac fibroblasts results in the recruitment of inflammatory Ly6Chi monocytes in pressure-overloaded hearts</div> <div class="author"> Hajime Abe , Yohei Tanada , Shigemiki Omiya , Mihai-Nicolae Podaru , Tomokazu Murakawa , Jumpei Ito , Ajay M Shah , Simon J Conway , <em>Masahiro Ono</em>, and Kinya Otsu </div> <div class="periodical"> <em>Science Signaling</em>, Jun 2021 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN65" class="col-sm-8"> <div class="title">Regulatory T cells restrain interleukin-2-and Blimp-1-dependent acquisition of cytotoxic function by CD4+ T cells</div> <div class="author"> Anna Śledzińska , Maria Vila Mucha , Katharina Bergerhoff , Alastair Hotblack , Dafne Franz Demane , Ehsan Ghorani , Ayse U Akarca , Maria AV Marzolini , Isabelle Solomon , and Frederick Arce Vargas </div> <div class="periodical"> <em>Immunity</em>, Jun 2020 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN66" class="col-sm-8"> <div class="title">Control of regulatory T-cell differentiation and function by T-cell receptor signalling and Foxp3 transcription factor complexes</div> <div class="author"> <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Immunology</em>, Jun 2020 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/imm.13178" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Summary The transcription factor Foxp3 controls the differentiation and function of regulatory T-cells (Treg). Studies in the past decades identified numerous Foxp3-interacting protein partners. However, it is still not clear how Foxp3 produces the Treg-type transcriptomic landscape through cooperating with its partners. Here I show the current understanding of how Foxp3 transcription factor complexes regulate the differentiation, maintenance and functional maturation of Treg. Importantly, T-cell receptor (TCR) signalling plays central roles in Treg differentiation and Foxp3-mediated gene regulation. Differentiating Treg will have recognized their cognate antigens and received TCR signals before initiating Foxp3 transcription, which is triggered by TCR-induced transcription factors including NFAT, AP-1 and NF-κB. Once expressed, Foxp3 seizes TCR signal-induced transcriptional and epigenetic mechanisms through interacting with AML1/Runx1 and NFAT. Thus, Foxp3 modifies gene expression dynamics of TCR-induced genes, which constitute cardinal mechanisms for Treg-mediated immune suppression. Next, I discuss the following key topics, proposing new mechanistic models for Foxp3-mediated gene regulation: (i) how Foxp3 transcription is induced and maintained by the Foxp3-inducing enhanceosome and the Foxp3 autoregulatory transcription factor complex; (ii) molecular mechanisms for effector Treg differentiation (i.e. Treg maturation); (iii) how Foxp3 activates or represses its target genes through recruiting coactivators and corepressors; (iv) the ‘decision-making’ Foxp3-containing transcription factor complex for Th17 and Treg differentiation; and (v) the roles of post-translational modification in Foxp3 regulation. Thus, this article provides cutting-edge understanding of molecular biology of Foxp3 and Treg, integrating findings by biochemical and genomic studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN68" class="col-sm-8"> <div class="title">T-cell hyperactivation and paralysis in severe COVID-19 infection revealed by single-cell analysis</div> <div class="author"> Bahire Kalfaoglu , José Almeida-Santos , Chanidapa Adele Tye , Yorifumi Satou , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Frontiers in immunology</em>, Jun 2020 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2020.589380/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Severe COVID-19 patients show various immunological abnormalities including T-cell reduction and cytokine release syndrome, which can be fatal and is a major concern of the pandemic. However, it is poorly understood how T-cell dysregulation can contribute to the pathogenesis of severe COVID-19. Here we show single cell-level mechanisms for T-cell dysregulation in severe COVID-19, demonstrating new pathogenetic mechanisms of T-cell activation and differentiation underlying severe COVID-19. By in silico sorting CD4+ T-cells from a single cell RNA-seq dataset, we found that CD4+ T-cells were highly activated and showed unique differentiation pathways in the lung of severe COVID-19 patients. Notably, those T-cells in severe COVID-19 patients highly expressed immunoregulatory receptors and CD25, whilst repressing the expression of FOXP3. Furthermore, we show that CD25+ hyperactivated T-cells differentiate into multiple helper T-cell lineages, showing multifaceted effector T-cells with Th1 and Th2 characteristics. Lastly, we show that CD25-expressing hyperactivated T-cells produce the protease Furin, which facilitates the viral entry of SARS-CoV-2. Collectively, CD4+ T-cells from severe COVID-19 patients are hyperactivated and FOXP3-mediated negative feedback mechanisms are impaired in the lung, which may promote immunopathology. Therefore, our study proposes a new model of T-cell hyperactivation and paralysis that drives immunopathology in severe COVID-19.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN71" class="col-sm-8"> <div class="title">Nr4a1 and Nr4a3 reporter mice are differentially sensitive to T cell receptor signal strength and duration</div> <div class="author"> Emma Jennings , Thomas AE Elliot , Natasha Thawait , Shivani Kanabar , Juan Carlos Yam-Puc , <em>Masahiro Ono</em>, Kai-Michael Toellner , David C Wraith , Graham Anderson , and David Bending </div> <div class="periodical"> <em>Cell reports</em>, Jun 2020 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN76" class="col-sm-8"> <div class="title">Mutant BRAF small molecule inhibition enhances oncolytic herpes virus immunotherapy through increased immune cell recruitment and activation in melanoma</div> <div class="author"> Galabina Bozhanova , Victoria Jennings , Malin Pedersen , Joan Kyula , Emmanuel Patin , Jehanne Hassan , <em>Masahiro Ono</em>, Kevin Harrington , and Alan Melcher </div> <div class="periodical"> <em>Cancer Research</em>, Jun 2020 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN59" class="col-sm-8"> <div class="title">IFITM proteins drive type 2 T helper cell differentiation and exacerbate allergic airway inflammation</div> <div class="author"> Diana C Yánez , Hemant Sahni , Susan Ross , Anisha Solanki , Ching‐In Lau , Eleftheria Papaioannou , Alessandro Barbarulo , Rebecca Powell , Ulrike C Lange , and David J Adams </div> <div class="periodical"> <em>European journal of immunology</em>, Jun 2019 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN63" class="col-sm-8"> <div class="title">Sonic Hedgehog signaling limits atopic dermatitis via Gli2-driven immune regulation</div> <div class="author"> Eleftheria Papaioannou , Diana C Yánez , Susan Ross , Ching-In Lau , Anisha Solanki , Mira Manilal Chawda , Alex Virasami , Ismael Ranz , <em>Masahiro Ono</em>, and Ryan FL O’Shaughnessy </div> <div class="periodical"> <em>The Journal of clinical investigation</em>, Jun 2019 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN60" class="col-sm-8"> <div class="title">Sonic hedgehog is a determinant of γδ T-cell differentiation in the thymus</div> <div class="author"> Konstantinos Mengrelis , Ching-In Lau , Jasmine Rowell , Anisha Solanki , Sonia Norris , Susan Ross , <em>Masahiro Ono</em>, Susan Outram , and Tessa Crompton </div> <div class="periodical"> <em>Frontiers in immunology</em>, Jun 2019 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN61" class="col-sm-8"> <div class="title">Distinct regulation of Nr4a receptors by NFAT and ERK signalling during T cell activation</div> <div class="author"> Emma Jennings , Shivani Kanabar , <em>Masahiro Ono</em>, David C Wraith , David Bending , and David Bending </div> <div class="periodical"> <em></em> Jun 2019 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN58" class="col-sm-8"> <div class="title">From stability to dynamics: understanding molecular mechanisms of regulatory T cells through Foxp3 transcriptional dynamics</div> <div class="author"> David Bending , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Clinical and Experimental Immunology</em>, Sep 2018 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1111/cei.13194" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Studies on regulatory T cells (Treg) have focused on thymic Treg as a stable lineage of immunosuppressive T cells, the differentiation of which is controlled by the transcription factor forkhead box protein 3 (Foxp3). This lineage perspective, however, may constrain hypotheses regarding the role of Foxp3 and Treg  in vivo, particularly in clinical settings and immunotherapy development. In this review, we synthesize a new perspective on the role of Foxp3 as a dynamically expressed gene, and thereby revisit the molecular mechanisms for the transcriptional regulation of Foxp3. In particular, we introduce a recent advancement in the study of Foxp3-mediated T cell regulation through the development of the Timer of cell kinetics and activity (Tocky) system, and show that the investigation of Foxp3 transcriptional dynamics can reveal temporal changes in the differentiation and function of Treg  in vivo. We highlight the role of Foxp3 as a gene downstream of T cell receptor (TCR) signalling and show that temporally persistent TCR signals initiate Foxp3 transcription in self-reactive thymocytes. In addition, we feature the autoregulatory transcriptional circuit for the Foxp3 gene as a mechanism for consolidating Treg differentiation and activating their suppressive functions. Furthermore, we explore the potential mechanisms behind the dynamic regulation of epigenetic modifications and chromatin architecture for Foxp3 transcription. Lastly, we discuss the clinical relevance of temporal changes in the differentiation and activation of Treg.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN57" class="col-sm-8"> <div class="title">Elucidating T Cell Activation-Dependent Mechanisms for Bifurcation of Regulatory and Effector T Cell Differentiation by Multidimensional and Single-Cell Analysis</div> <div class="author"> Alla Bradley , Tetsuo Hashimoto , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Frontiers in Immunology</em>, Sep 2018 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2018.01444" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p></p> <p>In T cells, T cell receptor (TCR) signaling initiates downstream transcriptional mechanisms for T cell activation and differentiation. Foxp3-expressing regulatory T cells (Treg) require TCR signals for their suppressive function and maintenance in the periphery. It is, however, unclear how TCR signaling controls the transcriptional program of Treg. Since most of studies identified the transcriptional features of Treg in comparison to naïve T cells, the relationship between Treg and non-naïve T cells including memory-phenotype T cells (Tmem) and effector T cells (Teff) is not well understood. Here, we dissect the transcriptomes of various T cell subsets from independent datasets using the multidimensional analysis method canonical correspondence analysis (CCA). We show that at the cell population level, resting Treg share gene modules for activation with Tmem and Teff. Importantly, Tmem activate the distinct transcriptional modules for T cell activation, which are uniquely repressed in Treg. The activation signature of Treg is dependent on TCR signals and is more actively operating in activated Treg. Furthermore, by using a new CCA-based method, single-cell combinatorial CCA, we analyzed unannotated single-cell RNA-seq data from tumor-infiltrating T cells, and revealed that FOXP3 expression occurs predominantly in activated T cells. Moreover, we identified FOXP3-driven and T follicular helper-like differentiation pathways in tumor microenvironments, and their bifurcation point, which is enriched with recently activated T cells. Collectively, our study reveals the activation mechanisms downstream of TCR signals for the bifurcation of Treg and Teff differentiation and their maturation processes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN56" class="col-sm-8"> <div class="title">A temporally dynamic Foxp3 autoregulatory transcriptional circuit controls the effector Treg programme</div> <div class="author"> David Bending , Alina Paduraru , Catherine B Ducker , Paz Prieto Martı́n , Tessa Crompton , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>The EMBO journal</em>, Sep 2018 </div> <div class="periodical"> <em style="color: darkgrey;">The second Tocky paper from the Ono lab uncovers the temporally dynamic regulation of Foxp3 transcription, offering new insights into T cell regulation.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.embopress.org/doi/full/10.15252/embj.201899013" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Regulatory T cells (Treg) are negative regulators of the immune response; however, it is poorly understood whether and how Foxp3 transcription is induced and regulated in the periphery during T‐cell responses. Using Foxp3‐Timer of cell kinetics and activity (Tocky) mice, which report real‐time Foxp3 expression, we show that the flux of new Foxp3 expressors and the rate of Foxp3 transcription are increased during inflammation. These persistent dynamics of Foxp3 transcription determine the effector Treg programme and are dependent on a Foxp3 autoregulatory transcriptional circuit. Persistent Foxp3 transcriptional activity controls the expression of coinhibitory molecules, including CTLA‐4 and effector Treg signature genes. Using RNA‐seq, we identify two groups of surface proteins based on their relationship to the temporal dynamics of Foxp3 transcription, and we show proof of principle for the manipulation of Foxp3 dynamics by immunotherapy: new Foxp3 flux is promoted by anti‐TNFRII antibody, and high‐frequency Foxp3 expressors are targeted by anti‐OX40 antibody. Collectively, our study dissects time‐dependent mechanisms behind Foxp3‐driven T‐cell regulation and establishes the Foxp3‐Tocky system as a tool to investigate the mechanisms behind T‐cell immunotherapies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN55" class="col-sm-8"> <div class="title">A timer for analyzing temporally dynamic changes in transcription during differentiation in vivo</div> <div class="author"> David Bending , Paz Prieto Martı́n , Alina Paduraru , Catherine Ducker , Erik Marzaganov , Marie Laviron , Satsuki Kitano , Hitoshi Miyachi , Tessa Crompton , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Journal of Cell Biology</em>, Sep 2018 </div> <div class="periodical"> <em style="color: darkgrey;">The foundational publication introducing Tocky technology by the Ono lab, marking a breakthrough in T cell and B cell studies.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://rupress.org/jcb/article/217/8/2931/39442/A-timer-for-analyzing-temporally-dynamic-changes" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Understanding the mechanisms of cellular differentiation is challenging because differentiation is initiated by signaling pathways that drive temporally dynamic processes, which are difficult to analyze in vivo. We establish a new tool, Timer of cell kinetics and activity (Tocky; or toki [time in Japanese]). Tocky uses the fluorescent Timer protein, which spontaneously shifts its emission spectrum from blue to red, in combination with computer algorithms to reveal the dynamics of differentiation in vivo. Using a transcriptional target of T cell receptor (TCR) signaling, we establish Nr4a3-Tocky to follow downstream effects of TCR signaling. Nr4a3-Tocky reveals the temporal sequence of events during regulatory T cell (Treg) differentiation and shows that persistent TCR signals occur during Treg generation. Remarkably, antigen-specific T cells at the site of autoimmune inflammation also show persistent TCR signaling. In addition, by generating Foxp3-Tocky, we reveal the in vivo dynamics of demethylation of the Foxp3 gene. Thus, Tocky is a tool for cell biologists to address previously inaccessible questions by directly revealing dynamic processes in vivo.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN54" class="col-sm-8"> <div class="title">Revealing the mechanism of regulatory T-cell generation in vivo by novel Fluorescent Timer reporter</div> <div class="author"> <em>Masahiro Ono</em>, David Bending , and Paz Prieto Martin </div> <div class="periodical"> <em>Journal of Immunology</em>, May 2017 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://journals.aai.org/jimmunol/article/198/1_Supplement/80.3/55245" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Foxp3+ regulatory T-cells (Treg) are considered to mainly differentiate in the thymus and protect against autoimmunity. However, the physiological significance of peripheral Treg differentiation is still unclear, due to a lack of methods to identify newly generated Treg in vivo. In addition, the mechanism of Treg differentiation in the thymus is still obscure, because of difficulties in defining temporal sequences of developmental events in vivo. Here we develop a ground-breaking technology to reveal the time-dependent mechanisms of Treg generation and maturation in vivo. Using Fluorescent Timer protein that spontaneously and irreversibly changes its emission spectrum, we establish a novel Foxp3Timer reporter, and and visualize the dynamics of Foxp3 transcription and thereby identify de novo Foxp3 expression in vivo. Here we show that antigenic recognition immediately induces de novo Foxp3 expression in activated effector T-cells in vivo. Surprisingly, most of Foxp3-expressors in inflamed tissue are not derived from the pre-existing Treg pool, but are newly generated upon antigen recognition. RNA-seq analysis reveals novel genomic mechanisms for peripheral Treg generation and identifies new markers for antigen-reactive Treg, by which we successfully manipulate Treg differentiation dynamics. Collectively, we show that Foxp3 constitutes a negative feedback mechanism for activated effector T-cells, which induces the resolution of inflammation. Our novel Timer technology is applicable to many outstanding problems in immunology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN52" class="col-sm-8"> <div class="title">The impact of environmental enrichment on the murine inflammatory immune response</div> <div class="author"> Samuel Brod , Thomas Gobbetti , Beatrice Gittens , <em>Masahiro Ono</em>, Mauro Perretti , and Fulvio D’Acquisto </div> <div class="periodical"> <em>JCI insight</em>, May 2017 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN53" class="col-sm-8"> <div class="title">FoxP3 partners up</div> <div class="author"> David Bending , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Nature Immunology</em>, May 2017 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN51" class="col-sm-8"> <div class="title">Interplay between the skin barrier and immune cells in patients with atopic dermatitis unraveled by means of mathematical modeling</div> <div class="author"> David Bending , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Journal of Allergy and Clinical Immunology</em>, May 2017 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN49" class="col-sm-8"> <div class="title">Water resistance profile as a marker of skin barrier damage in atopic dermatitis patients</div> <div class="author"> Mark DA Logtestijn , Peter J Caspers , Sanja Kezic , Douglas R Hoffman , David W Koenig , <em>Masahiro Ono</em>, Georgios N Stamatas , and Reiko J Tanaka </div> <div class="periodical"> <em>Journal of Dermatological Science</em>, May 2016 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN48" class="col-sm-8"> <div class="title">Sonic Hedgehog regulates thymic epithelial cell differentiation</div> <div class="author"> José Ignacio Saldaña , Anisha Solanki , Ching-In Lau , Hemant Sahni , Susan Ross , Anna L Furmanski , <em>Masahiro Ono</em>, Georg Holländer , and Tessa Crompton </div> <div class="periodical"> <em>Journal of Autoimmunity</em>, May 2016 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN50" class="col-sm-8"> <div class="title">Impact of enriched environment on murine T cell differentiation and gene expression profile</div> <div class="author"> Lorenza Rattazzi , Giuseppa Piras , Samuel Brod , Koval Smith , <em>Masahiro Ono</em>, and Fulvio D’Acquisto </div> <div class="periodical"> <em>Frontiers in Immunology</em>, May 2016 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN44" class="col-sm-8"> <div class="title">Controversies concerning thymus‐derived regulatory T cells: fundamental issues and a new perspective</div> <div class="author"> <em>Masahiro Ono</em>, and Reiko J Tanaka </div> <div class="periodical"> <em>Immunology and cell biology</em>, May 2016 </div> <div class="periodical"> <em style="color: darkgrey;">A landmark opinion piece challenging the reproducibility of a foundational Treg experiment, while introducing a groundbreaking dynamic view on Foxp3-mediated T cell regulation.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://onlinelibrary.wiley.com/doi/10.1038/icb.2015.65" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Thymus-derived regulatory T cells (Tregs) are considered to be a distinct T-cell lineage that is genetically programmed and specialised for immunosuppression. This perspective is based on the key evidence that CD25+ Tregs emigrate to neonatal spleen a few days later than other T cells and that thymectomy of 3-day-old mice depletes Tregs only, causing autoimmune diseases. Although widely believed, the evidence has never been reproduced as originally reported, and some studies indicate that Tregs exist in neonates. Thus we examine the consequences of the controversial evidence, revisit the fundamental issues of Tregs and thereby reveal the overlooked relationship of T-cell activation and Foxp3-mediated control of the T-cell system. Here we provide a new model of Tregs and Foxp3, a feedback control perspective, which views Tregs as a component of the system that controls T-cell activation, rather than as a distinct genetically programmed lineage. This perspective provides new insights into the roles of self-reactivity, T cell–antigen-presenting cell interaction and T-cell activation in Foxp3-mediated immune regulation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN47" class="col-sm-8"> <div class="title">Regulatory T cells in melanoma revisited by a computational clustering of FOXP3+ T cell subpopulations</div> <div class="author"> Hiroko Fujii , Julie Josse , Miki Tanioka , Yoshiki Miyachi , François Husson , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>The Journal of Immunology</em>, May 2016 </div> <div class="periodical"> <em style="color: darkgrey;">The pioneering study ntroduces a data-driven methodology to explore Treg subpopulations, employing Principal Component Analysis (PCA) for the first time.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://journals.aai.org/jimmunol/article/196/6/2885/111920/Regulatory-T-Cells-in-Melanoma-Revisited-by-a" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>CD4+ T cells that express the transcription factor FOXP3 (FOXP3+ T cells) are commonly regarded as immunosuppressive regulatory T cells (Tregs). FOXP3+ T cells are reported to be increased in tumor-bearing patients or animals and are considered to suppress antitumor immunity, but the evidence is often contradictory. In addition, accumulating evidence indicates that FOXP3 is induced by antigenic stimulation and that some non-Treg FOXP3+ T cells, especially memory-phenotype FOXP3low cells, produce proinflammatory cytokines. Accordingly, the subclassification of FOXP3+ T cells is fundamental for revealing the significance of FOXP3+ T cells in tumor immunity, but the arbitrariness and complexity of manual gating have complicated the issue. In this article, we report a computational method to automatically identify and classify FOXP3+ T cells into subsets using clustering algorithms. By analyzing flow cytometric data of melanoma patients, the proposed method showed that the FOXP3+ subpopulation that had relatively high FOXP3, CD45RO, and CD25 expressions was increased in melanoma patients, whereas manual gating did not produce significant results on the FOXP3+ subpopulations. Interestingly, the computationally identified FOXP3+ subpopulation included not only classical FOXP3high Tregs, but also memory-phenotype FOXP3low cells by manual gating. Furthermore, the proposed method successfully analyzed an independent data set, showing that the same FOXP3+ subpopulation was increased in melanoma patients, validating the method. Collectively, the proposed method successfully captured an important feature of melanoma without relying on the existing criteria of FOXP3+ T cells, revealing a hidden association between the T cell profile and melanoma, and providing new insights into FOXP3+ T cells and Tregs.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN43" class="col-sm-8"> <div class="title">A Zap70‐dependent feedback circuit is essential for efficient selection of CD4 lineage thymocytes</div> <div class="author"> Charles Sinclair , <em>Masahiro Ono</em>, and Benedict Seddon </div> <div class="periodical"> <em>Immunology and Cell Biology</em>, May 2015 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN46" class="col-sm-8"> <div class="title">A genome wide transcriptional model of the complex response to pre-TCR signalling during thymocyte differentiation</div> <div class="author"> Hemant Sahni , Susan Ross , Alessandro Barbarulo , Anisha Solanki , Ching-In Lau , Anna Furmanski , Jose Ignacio Saldana , <em>Masahiro Ono</em>, Mike Hubank , and Martino Barenco </div> <div class="periodical"> <em>Oncotarget</em>, May 2015 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="JCI76238" class="col-sm-8"> <div class="title">Follicular helper T cell signature in type 1 diabetes</div> <div class="author"> Rupert Kenefeck , Chun Jing Wang , Tauseef Kapadi , Lukasz Wardzinski , Kesley Attridge , Louise E. Clough , Frank Heuts , Alexandros Kogimtzis , Sapna Patel , Miranda Rosenthal , <em>Masahiro Ono</em>, David M. Sansom , Parth Narendran , and Lucy S.K. Walker </div> <div class="periodical"> <em>The Journal of Clinical Investigation</em>, Jan 2015 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1172/JCI76238" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>The strong genetic association between particular HLA alleles and type 1 diabetes (T1D) indicates a key role for CD4+ T cells in disease; however, the differentiation state of the responsible T cells is unclear. T cell differentiation originally was considered a dichotomy between Th1 and Th2 cells, with Th1 cells deemed culpable for autoimmune islet destruction. Now, multiple additional T cell differentiation fates are recognized with distinct roles. Here, we used a transgenic mouse model of diabetes to probe the gene expression profile of islet-specific T cells by microarray and identified a clear follicular helper T (Tfh) cell differentiation signature. Introduction of T cells with a Tfh cell phenotype from diabetic animals efficiently transferred diabetes to recipient animals. Furthermore, memory T cells from patients with T1D expressed elevated levels of Tfh cell markers, including CXCR5, ICOS, PDCD1, BCL6, and IL21. Defects in the IL-2 pathway are associated with T1D, and IL-2 inhibits Tfh cell differentiation in mice. Consistent with these previous observations, we found that IL-2 inhibited human Tfh cell differentiation and identified a relationship between IL-2 sensitivity in T cells from patients with T1D and acquisition of a Tfh cell phenotype. Together, these findings identify a Tfh cell signature in autoimmune diabetes and suggest that this population could be used as a biomarker and potentially targeted for T1D interventions.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN38" class="col-sm-8"> <div class="title">Identifying a hyperkeratosis signature in autosomal recessive congenital ichthyosis: Mdm2 inhibition prevents hyperkeratosis in a rat ARCI model</div> <div class="author"> Gehad Youssef , <em>Masahiro Ono</em>, Stuart J Brown , Veronica A Kinsler , Neil J Sebire , John I Harper , and Ryan FL O’shaughnessy </div> <div class="periodical"> <em>The Journal of Investigative Dermatology</em>, Jan 2014 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN97" class="col-sm-8"> <div class="title">Visualisation of the T cell differentiation programme by Canonical Correspondence Analysis of transcriptomes</div> <div class="author"> <em>Masahiro Ono</em>, Reiko Tanaka , and Manabu Kano </div> <div class="periodical"> <em>BMC Genomics</em>, Jan 2014 </div> <div class="periodical"> <em style="color: darkgrey;">This study establishes CCA as a vital quantitative tool for transcriptome analysis in immunological datasets.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/25428805" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>BACKGROUND: Currently, in the era of post-genomics, immunology is facing a challenging problem to translate mutant phenotypes into gene functions based on high-throughput data, while taking into account the classifications and functions of immune cells, which requires new methods. RESULTS: Here we propose a novel application of a multidimensional analysis, Canonical Correspondence Analysis (CCA), to reveal the molecular characteristics of undefined cells in terms of cellular differentiation programmes by analysing two transcriptomic datasets. Using two independent datasets, whether RNA-seq or microarray data, CCA successfully visualised the cross-level relationships between genes, cells, and differentiation programmes, and thereby identified the immunological features of mutant cells (Gata3-KO T cells and Stat3-KO T cells) in a data-oriented manner. With a new concept, differentiation variable, CCA provides an automatic classification of cell samples, which had a high sensitivity and a comparable performance to other classification methods. In addition, we elaborate how CCA results can be interpreted, and reveal the features of CCA in comparison with other visualisation techniques. CONCLUSIONS: CCA is a visualisation tool with a classification ability to reveal the cross-level relationships of genes, cells and differentiation programmes. This can be used for characterising the functional defect of cells of interest (e.g. mutant cells) in the context of cellular differentiation. The proposed approach fits with common hypothesis-oriented studies in immunology, and can be used for a wide range of molecular and genomic studies on cellular differentiation mechanisms.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN40" class="col-sm-8"> <div class="title">CD8+ tumor‐infiltrating lymphocytes at primary sites as a possible prognostic factor of cutaneous angiosarcoma</div> <div class="author"> Hiroko Fujii , Akiko Arakawa , Daisuke Utsumi , Shinji Sumiyoshi , Yosuke Yamamoto , Akihiko Kitoh , <em>Masahiro Ono</em>, Yumi Matsumura , Mayumi Kato , and Keisuke Konishi </div> <div class="periodical"> <em>International journal of cancer</em>, Jan 2014 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN39" class="col-sm-8"> <div class="title">Rab3Gap1 mediates exocytosis of Claudin-1 and tight junction formation during epidermal barrier acquisition</div> <div class="author"> G Youssef , L Gerner , AS Naeem , O Ralph , M Ono , CA O’neill , and RFL O’Shaughnessy </div> <div class="periodical"> <em>Developmental biology</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN36" class="col-sm-8"> <div class="title">Skin disease modeling from a mathematical perspective</div> <div class="author"> Reiko J Tanaka , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Journal of Investigative Dermatology</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN37" class="col-sm-8"> <div class="title">CD4+ but not CD8+ T cells revert the impaired emotional behavior of immunocompromised RAG-1-deficient mice</div> <div class="author"> Lorenza Rattazzi , Giuseppa Piras , Mutsumi Ono , Robert Deacon , CM Pariante , and F D’acquisto </div> <div class="periodical"> <em>Translational psychiatry</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN98" class="col-sm-8"> <div class="title">Visualising the cross-level relationships between pathological and physiological processes and gene expression: analyses of haematological diseases</div> <div class="author"> <em>Masahiro Ono</em>, Reiko Tanaka , Manabu Kano , and Toshio Sugiman </div> <div class="periodical"> <em>PLoS One</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;">The pioneering study developing Canonical Correspondence Analysis (CCA) as a genomics method, establishing a novel approach for transcriptome analysis.</em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/23301083" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>The understanding of pathological processes is based on the comparison between physiological and pathological conditions, and transcriptomic analysis has been extensively applied to various diseases for this purpose. However, the way in which the transcriptomic data of pathological cells relate to the transcriptomes of normal cellular counterparts has not been fully explored, and may provide new and unbiased insights into the mechanisms of these diseases. To achieve this, it is necessary to develop a method to simultaneously analyse components across different levels, namely genes, normal cells, and diseases. Here we propose a multidimensional method that visualises the cross-level relationships between these components at three different levels based on transcriptomic data of physiological and pathological processes, by adapting Canonical Correspondence Analysis, which was developed in ecology and sociology, to microarray data (CCA on Microarray data, CCAM). Using CCAM, we have analysed transcriptomes of haematological disorders and those of normal haematopoietic cell differentiation. First, by analysing leukaemia data, CCAM successfully visualised known relationships between leukaemia subtypes and cellular differentiation, and their characteristic genes, which confirmed the relevance of CCAM. Next, by analysing transcriptomes of myelodysplastic syndromes (MDS), we have shown that CCAM was effective in both generating and testing hypotheses. CCAM showed that among MDS patients, high-risk patients had transcriptomes that were more similar to those of both haematopoietic stem cells (HSC) and megakaryocyte-erythroid progenitors (MEP) than low-risk patients, and provided a prognostic model. Collectively, CCAM reveals hidden relationships between pathological and physiological processes and gene expression, providing meaningful clinical insights into haematological diseases, and these could not be revealed by other univariate and multivariate methods. Furthermore, CCAM was effective in identifying candidate genes that are correlated with cellular phenotypes of interest. We expect that CCAM will benefit a wide range of medical fields.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN35" class="col-sm-8"> <div class="title">Tissue-Derived Hedgehog Proteins Modulate Th Differentiation and Disease</div> <div class="author"> Anna L Furmanski , Jose Ignacio Saldana , <em>Masahiro Ono</em>, Hemant Sahni , Nikolaos Paschalidis , Fulvio D’Acquisto , and Tessa Crompton </div> <div class="periodical"> <em>The Journal of Immunology</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN34" class="col-sm-8"> <div class="title">Risk factor-dependent dynamics of atopic dermatitis: modelling multi-scale regulation of epithelium homeostasis</div> <div class="author"> Elisa Domı́nguez-Hüttinger , <em>Masahiro Ono</em>, Mauricio Barahona , and Reiko J Tanaka </div> <div class="periodical"> <em>Interface Focus</em>, Jan 2013 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN30" class="col-sm-8"> <div class="title">Differential effects of inhibition of bone morphogenic protein (BMP) signalling on T-cell activation and differentiation</div> <div class="author"> Y Yoshioka , M Ono , M Osaki , I Konishi , and S Sakaguchi </div> <div class="periodical"> <em>Eur J Immunol</em>, Jan 2011 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://onlinelibrary.wiley.com/doi/10.1002/eji.201141702" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Bone morphogenetic proteins (BMPs) are involved in patterning and cellular fate in various organs including the thymus. However, the redundancy of BMPs and their receptors have made it difficult to analyse their physiological roles. Here, we investigated the role of BMP signalling in peripheral CD4(+) T cells by analysing the effects of an inhibitor of BMP signalling, dorsomorphin. Dorsomorphin suppressed phosphorylation of SMAD1/5/8, suggesting that BMP signalling naturally occurs in T cells. At high doses, dorsomorphin suppressed proliferation of T cells in a dose-dependent manner, inducing G1 arrest. Also, dorsomorphin suppressed Th17 and induced Treg-cell differentiation, while preserving Th2 differentiation. Dorsomorphin efficiently suppressed IL-2 production even at low doses in mouse CD4(+) T cells, suggesting that the BMP-Smad signalling physiologically regulates IL-2 transcription in these cells. In addition, recombinant BMP2 induced a dose-dependent multiphasic pattern of IL-2 production, while Noggin suppressed IL-2 production at higher doses in Jurkat cells. Notably, BMP signalling controlled the phosphorylation of RUNX1, revealing the molecular nature of its effect. Collectively, we describe multiple effects of dorsomorphin and Noggin on T-cell activation and differentiation, demonstrating a physiological role for BMP signalling in these processes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN31" class="col-sm-8"> <div class="title">Skin barrier homeostasis in atopic dermatitis: feedback regulation of kallikrein activity</div> <div class="author"> Reiko J Tanaka , <em>Masahiro Ono</em>, and Heather A Harrington </div> <div class="periodical"> <em>PLoS One</em>, Jan 2011 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="10.1371/journal.ppat.1001274" class="col-sm-8"> <div class="title">HTLV-1 bZIP Factor Induces T-Cell Lymphoma and Systemic Inflammation In Vivo</div> <div class="author"> Yorifumi Satou , Jun-ichirou Yasunaga , Tiejun Zhao , Mika Yoshida , Paola Miyazato , Ken Takai , Kei Shimizu , Koichi Ohshima , Patrick L. Green , Naganari Ohkura , Tomoyuki Yamaguchi , <em>Masahiro Ono</em>, Shimon Sakaguchi , and Masao Matsuoka </div> <div class="periodical"> <em>PLOS Pathogens</em>, Feb 2011 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1371/journal.ppat.1001274" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Human T-cell leukemia virus type 1 (HTLV-1) is the causal agent of a neoplastic disease of CD4+ T cells, adult T-cell leukemia (ATL), and inflammatory diseases including HTLV-1 associated myelopathy/tropical spastic paraparesis, dermatitis, and inflammatory lung diseases. ATL cells, which constitutively express CD25, resemble CD25+CD4+ regulatory T cells (Treg). Approximately 60% of ATL cases indeed harbor leukemic cells that express FoxP3, a key transcription factor for Treg cells. HTLV-1 encodes an antisense transcript, HTLV-1 bZIP factor (HBZ), which is expressed in all ATL cases. In this study, we show that transgenic expression of HBZ in CD4+ T cells induced T-cell lymphomas and systemic inflammation in mice, resembling diseases observed in HTLV-1 infected individuals. In HBZ-transgenic mice, CD4+Foxp3+ Treg cells and effector/memory CD4+ T cells increased in vivo. As a mechanism of increased Treg cells, HBZ expression directly induced Foxp3 gene transcription in T cells. The increased CD4+Foxp3+ Treg cells in HBZ transgenic mice were functionally impaired while their proliferation was enhanced. HBZ could physically interact with Foxp3 and NFAT, thereby impairing the suppressive function of Treg cells. Thus, the expression of HBZ in CD4+ T cells is a key mechanism of HTLV-1-induced neoplastic and inflammatory diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN29" class="col-sm-8"> <div class="title">Perturbations of both nonregulatory and regulatory FOXP3+ T cells in patients with malignant melanoma</div> <div class="author"> Hiroko Fujii , A Arakawa , A Kitoh , M Miyara , M Kato , S Kore‐Eda , S Sakaguchi , Y Miyachi , M Tanioka , and M Ono </div> <div class="periodical"> <em>British Journal of Dermatology</em>, Feb 2011 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://academic.oup.com/bjd/article-abstract/164/5/1052/6642984" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Background: ’FOXP3+ regulatory T cells’ (Tregs) are reported to be increased in tumour-bearing hosts including patients with melanoma, leading to tumour immune suppression. However, this idea is challenged by recent evidence that the ’FOXP3+ Treg’ fraction in fact contains activated ’nonregulatory’ T cells. Also, FOXP3+ T cells are reported to have functionally and kinetically distinct subsets. Objectives: To investigate whether either or both of regulatory and ’nonregulatory’ FOXP3+ T cells are perturbed in patients with melanoma. Methods: FOXP3+ T cells were classified into three subsets, namely CD45RO+FOXP3(low) nonregulatory T cells, CD45RO+FOXP3(high) effector Tregs, and CD45RO-FOXP3(low) naïve Tregs, according to their expression levels of FOXP3 and CD45RO. The percentage and cytokine production of these FOXP3+ T-cell subsets were assessed by flow cytometry. Results: Both regulatory and nonregulatory T cells were increased in patients with melanoma. Moreover, we found three unexpected perturbations in FOXP3+ T-cell subsets: (i) patients with melanoma showed higher frequencies of FOXP3(low) nonregulatory T cells, which decreased and normalized after tumour removal; (ii) FOXP3(low) naïve Tregs containing higher frequencies of interferon-γ+ cells increased with tumour progression; and (iii) CD45RO+FOXP3(high) effector Tregs were pronouncedly infiltrated around tumour tissues. Conclusions: These findings demonstrate that patients with melanoma have distinct and differential perturbation of both regulatory and nonregulatory FOXP3+ T cells. The degree of perturbation is associated with tumour burden and progression, suggesting that the perturbation reflects fundamental pathophysiological processes in patients with melanoma. The presented analysis provides a practical approach to investigate the immunological environment of cancer patients.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN24" class="col-sm-8"> <div class="title">Functional Delineation and Differentiation Dynamics of Human CD4<sup>+</sup> T Cells Expressing the FoxP3 Transcription Factor</div> <div class="author"> Makoto Miyara , Yumiko Yoshioka , Akihiko Kitoh , Tomoko Shima , Kajsa Wing , Akira Niwa , Christophe Parizot , Cécile Taflin , Toshio Heike , Dominique Valeyre , Alexis Mathian , Tatsutoshi Nakahata , Tomoyuki Yamaguchi , Takashi Nomura , <em>Masahiro Ono</em>, Zahir Amoura , Guy Gorochov , and Shimon Sakaguchi </div> <div class="periodical"> <em>Immunity</em>, Feb 2009 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1016/j.immuni.2009.03.019" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>FoxP3 is a key transcription factor for the development and function of natural CD4+ regulatory T cells (Treg cells). Here we show that human FoxP3+CD4+ T cells were composed of three phenotypically and functionally distinct subpopulations: CD45RA+FoxP3lo resting Treg cells (rTreg cells) and CD45RA?FoxP3hi activated Treg cells (aTreg cells), both of which were suppressive in vitro, and cytokine-secreting CD45RA?FoxP3lo nonsuppressive T cells. The proportion of the three subpopulations differed between cord blood, aged individuals, and patients with immunological diseases. Terminally differentiated aTreg cells rapidly died whereas rTreg cells proliferated and converted into aTreg cells in vitro and in vivo. This was shown by the transfer of rTreg cells into NOD-scid-common ?-chain-deficient mice and by TCR sequence-based T cell clonotype tracing in peripheral blood in a normal individual. Taken together, the dissection of FoxP3+ cells into subsets enables one to analyze Treg cell differentiation dynamics and interactions in normal and disease states, and to control immune responses through manipulating particular FoxP3+ subpopulations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN26" class="col-sm-8"> <div class="title">Indispensable Role of the Runx1-Cbfβ Transcription Complex for In Vivo-Suppressive Function of FoxP3<sup>+</sup> Regulatory T Cells</div> <div class="author"> Akihiko Kitoh , <em>Masahiro Ono</em>, Yoshinori Naoe , Naganari Ohkura , Tomoyuki Yamaguchi , Hiroko Yaguchi , Issay Kitabayashi , Toshihiko Tsukada , Takashi Nomura , Yoshiki Miyachi , Ichiro Taniuchi , and Shimon Sakaguchi </div> <div class="periodical"> <em>Immunity</em>, Feb 2009 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1016/j.immuni.2009.09.003" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Naturally arising regulatory T (Treg) cells express the transcription factor FoxP3, which critically controls the development and function of Treg cells. FoxP3 interacts with another transcription factor Runx1 (also known as AML1). Here, we showed that Treg cell-specific deficiency of Cbf?, a cofactor for all Runx proteins, or that of Runx1, but not Runx3, induced lymphoproliferation, autoimmune disease, and hyperproduction of IgE. Cbfb-deleted Treg cells exhibited impaired suppressive function in vitro and in vivo, with altered gene expression profiles including attenuated expression of FoxP3 and high expression of interleukin-4. The Runx complex bound to more than 3000 gene loci in Treg cells, including the Foxp3 regulatory regions and the Il4 silencer. In addition, knockdown of RUNX1 showed that RUNX1 is required for the optimal regulation of FoxP3 expression in human T cells. Taken together, our results indicate that the Runx1-Cbf? heterodimer is indispensable for in vivo Treg cell function, in particular, suppressive activity and optimal expression of FoxP3.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN21" class="col-sm-8"> <div class="title">Regulatory T Cells and Immune Tolerance</div> <div class="author"> Shimon Sakaguchi , Tomoyuki Yamaguchi , Takashi Nomura , and <em>Masahiro Ono</em> </div> <div class="periodical"> <em>Cell</em>, Feb 2008 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://doi.org/10.1016/j.cell.2008.05.009" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Regulatory T cells (Tregs) play an indispensable role in maintaining immunological unresponsiveness to self-antigens and in suppressing excessive immune responses deleterious to the host. Tregs are produced in the thymus as a functionally mature subpopulation of T cells and can also be induced from naive T cells in the periphery. Recent research reveals the cellular and molecular basis of Treg development and function and implicates dysregulation of Tregs in immunological disease.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN23" class="col-sm-8"> <div class="title">Foxp3 controls regulatory T-cell function by interacting with AML1/Runx1</div> <div class="author"> <em>Masahiro Ono</em>, Hiroko Yaguchi , Naganari Ohkura , Issay Kitabayashi , Yuko Nagamura , Takashi Nomura , Yoshiki Miyachi , Toshihiko Tsukada , and Shimon Sakaguchi </div> <div class="periodical"> <em>Nature</em>, Feb 2007 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://www.nature.com/articles/nature05673" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Naturally arising CD25+CD4+ regulatory T cells (T(R) cells) are engaged in the maintenance of immunological self-tolerance and immune homeostasis by suppressing aberrant or excessive immune responses, such as autoimmune disease and allergy. T(R) cells specifically express the transcription factor Foxp3, a key regulator of T(R)-cell development and function. Ectopic expression of Foxp3 in conventional T cells is indeed sufficient to confer suppressive activity, repress the production of cytokines such as interleukin-2 (IL-2) and interferon-gamma (IFN-gamma), and upregulate T(R)-cell-associated molecules such as CD25, cytotoxic T-lymphocyte-associated antigen-4, and glucocorticoid-induced TNF-receptor-family-related protein. However, the method by which Foxp3 controls these molecular events has yet to be explained. Here we show that the transcription factor AML1 (acute myeloid leukaemia 1)/Runx1 (Runt-related transcription factor 1), which is crucially required for normal haematopoiesis including thymic T-cell development, activates IL-2 and IFN-gamma gene expression in conventional CD4+ T cells through binding to their respective promoters. In natural T(R) cells, Foxp3 interacts physically with AML1. Several lines of evidence support a model in which the interaction suppresses IL-2 and IFN-gamma production, upregulates T(R)-cell-associated molecules, and exerts suppressive activity. This transcriptional control of T(R)-cell function by an interaction between Foxp3 and AML1 can be exploited to control physiological and pathological T-cell-mediated immune responses.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2006</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN22" class="col-sm-8"> <div class="title">Foxp3+ CD25+ CD4+ natural regulatory T cells in dominant self‐tolerance and autoimmune disease</div> <div class="author"> Shimon Sakaguchi , <em>Masahiro Ono</em>, Ruka Setoguchi , Haruhiko Yagi , Shohei Hori , Zoltan Fehervari , Jun Shimizu , Takeshi Takahashi , and Takashi Nomura </div> <div class="periodical"> <em>Immunological Reviews</em>, Feb 2006 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN25" class="col-sm-8"> <div class="title">Control of autoimmune myocarditis and multiorgan inflammation by glucocorticoid-induced TNF receptor family-related proteinhigh, Foxp3-expressing CD25+ and CD25− regulatory T cells</div> <div class="author"> <em>Masahiro Ono</em>, Jun Shimizu , Yoshiki Miyachi , and Shimon Sakaguchi </div> <div class="periodical"> <em>The Journal of Immunology</em>, Feb 2006 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="https://journals.aai.org/jimmunol/article/176/8/4748/37234/Control-of-Autoimmune-Myocarditis-and-Multiorgan" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Publication</a> </div> <div class="abstract hidden"> <p>Anomalies of naturally occurring CD4+ regulatory T cells (Treg) cause severe autoimmune/inflammatory diseases in humans and rodents. The transcription factor Foxp3 is currently the most specific marker for natural CD4+ Treg, but it would be useful if other Treg markers, particularly cell surface molecules, could be elucidated. We demonstrate in this study that the vast majority of Foxp3-expressing CD4+ T cells (whether CD25+ or CD25−) show constitutive high-level expression of glucocorticoid-induced TNFR family-related gene/protein (GITR). Transfer of T cell or thymocyte suspensions depleted of GITRhigh cells produces in BALB/c nude mice a wider spectrum and more severe forms of autoimmune diseases than does transfer of similar cell suspensions depleted of CD25+CD4+ T cells only. Notably, mice that receive cells depleted of GITRhigh populations develop severe multiorgan inflammation that includes fatal autoimmune myocarditis resembling giant cell myocarditis in humans, accompanying high-titer anti-myosin autoantibodies. Similar transfer of GITRhigh-depleted cells from prediabetic NOD mice to NOD-SCID mice accelerates the development of diabetes and induces skeletal muscle myositis and other autoimmune/inflammatory diseases. We conclude that GITRhigh, Foxp3-expressing natural Treg, containing both CD25+ and CD25− cell populations, contribute to preventing a variety of autoimmune/inflammatory diseases, and depletion of these cells allows the activation of even weak or rare autoreactive T cells yielding widespread severe autoimmune disease. Diseases induced in this way include many which have been suspected of an autoimmune etiology in humans without much evidence. GITRhigh, Foxp3-expressing natural Treg represent a potential target for the treatment and prevention of these diseases.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2004</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="RN28" class="col-sm-8"> <div class="title">Cutaneous alternariosis in an immunocompetent patient: analysis of the internal transcribed spacer region of rDNA and Brm2 of isolated Alternaria alternata</div> <div class="author"> M Ono , C Nishigori , C Tanaka , S Tanaka , M Tsuda , and Y Miyachi </div> <div class="periodical"> <em>British Journal of Dermatology</em>, Feb 2004 </div> <div class="periodical"> <em style="color: darkgrey;"></em> </div> <div class="links"> </div> </div> </div> </li></ol> </div> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication/bookroom2-480.webp 480w,/assets/img/publication/bookroom2-800.webp 800w,/assets/img/publication/bookroom2-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/publication/bookroom2.jpg" width="300" height="auto" data-zoomable="" loading="lazy" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Masahiro Ono. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: April 17, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?5d75c11f89cd96294bf5e6dd1ee1bb30"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-VJHT0Y5RYF"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-VJHT0Y5RYF");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>